cmd/builder/internal/builder/main.go

```diff
@@ -13,13 +13,26 @@ import (
 	"time"
 
 	"go.uber.org/zap"
+	"go.uber.org/zap/zapio"
 )
 
 var (
 	// ErrGoNotFound is returned when a Go binary hasn't been found
 	ErrGoNotFound = errors.New("go binary not found")
 )
 
+func runGoCommand(cfg Config, args ...string) error {
+	cfg.Logger.Info("Running go subcommand.", zap.Any("arguments", args))
+	// #nosec G204 -- cfg.Distribution.Go is trusted to be a safe path and the caller is assumed to have carried out necessary input validation
+	cmd := exec.Command(cfg.Distribution.Go, args...)
+	cmd.Dir = cfg.Distribution.OutputPath
+	writer := &zapio.Writer{Log: cfg.Logger}
+	defer func() { _ = writer.Close() }()
+	cmd.Stdout = writer
```

Comment:
Any thoughts on how noisy this becomes? I wonder if it should be configurable

Reply:
We can take contrib as a worst-case scenario. The logging can be quite verbose in this case, it entirely depends on how many modules you have in your Go modules cache. Here are example logs for the contrib v0.87.0 manifest with the current version on main and with the proposed change, first with a clean module cache, then a re-run:

<details>
<summary> contrib v0.87.0 manifest, no logging from go subcommands (builder version from main)</summary> 

```
❯ ./builder --config manifest.yaml 
2023-10-23T12:12:50.119+0200    INFO    internal/command.go:121 OpenTelemetry Collector Builder {"version": "dev", "date": "unknown"}
2023-10-23T12:12:50.131+0200    INFO    internal/command.go:157 Using config file       {"path": "manifest.yaml"}
2023-10-23T12:12:50.131+0200    INFO    builder/config.go:108   Using go        {"go-executable": "/home/pablo.baeyens/.gvm/gos/go1.20/bin/go"}
2023-10-23T12:12:50.143+0200    INFO    builder/main.go:69      Sources created {"path": "./_build"}
2023-10-23T12:12:55.618+0200    INFO    builder/main.go:121     Getting go modules
2023-10-23T12:13:07.068+0200    INFO    builder/main.go:80      Compiling
2023-10-23T12:15:37.076+0200    INFO    builder/main.go:102     Compiled        {"binary": "./_build/otelcol-contrib"}
```

</details>

<details>
<summary>contrib v0.87.0 manifest after cleaning the module cache</summary>

```
❯ ./builder --config manifest.yaml 
2023-10-23T12:19:11.700+0200    INFO    internal/command.go:117 OpenTelemetry Collector Builder {"version": "dev", "date": "unknown"}
2023-10-23T12:19:11.712+0200    INFO    internal/command.go:153 Using config file       {"path": "manifest.yaml"}
2023-10-23T12:19:11.712+0200    INFO    builder/config.go:106   Using go        {"go-executable": "/home/pablo.baeyens/.gvm/gos/go1.20/bin/go"}
2023-10-23T12:19:11.722+0200    INFO    builder/main.go:82      Sources created {"path": "./_build"}
2023-10-23T12:19:11.722+0200    INFO    builder/main.go:25      Running go subcommand.  {"arguments": ["mod", "tidy", "-compat=1.20"]}
2023-10-23T12:19:11.734+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/exporter/awsemfexporter v0.87.0
2023-10-23T12:19:11.734+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/exporter/cassandraexporter v0.87.0
2023-10-23T12:19:11.734+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/exporter/datasetexporter v0.87.0
2023-10-23T12:19:11.734+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/connector/routingconnector v0.87.0
2023-10-23T12:19:11.735+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/exporter/clickhouseexporter v0.87.0
2023-10-23T12:19:11.735+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/connector/countconnector v0.87.0
2023-10-23T12:19:11.735+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/exporter/azuremonitorexporter v0.87.0
2023-10-23T12:19:11.736+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/connector/datadogconnector v0.87.0
2023-10-23T12:19:11.736+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/exporter/carbonexporter v0.87.0
2023-10-23T12:19:11.736+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/connector/spanmetricsconnector v0.87.0
2023-10-23T12:19:11.736+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/connector/servicegraphconnector v0.87.0
2023-10-23T12:19:11.737+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/exporter/datadogexporter v0.87.0
2023-10-23T12:19:11.737+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/exporter/alibabacloudlogserviceexporter v0.87.0
2023-10-23T12:19:11.737+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/exporter/awscloudwatchlogsexporter v0.87.0
2023-10-23T12:19:11.737+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/exporter/coralogixexporter v0.87.0
2023-10-23T12:19:11.740+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/exporter/awsxrayexporter v0.87.0
2023-10-23T12:19:11.740+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/exporter/awss3exporter v0.87.0
2023-10-23T12:19:11.740+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/exporter/azuredataexplorerexporter v0.87.0
2023-10-23T12:19:11.742+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/exporter/awskinesisexporter v0.87.0
2023-10-23T12:19:11.744+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/exporter/dynatraceexporter v0.87.0
2023-10-23T12:19:11.884+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/exporter/elasticsearchexporter v0.87.0
2023-10-23T12:19:11.885+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/exporter/f5cloudexporter v0.87.0
2023-10-23T12:19:11.885+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/exporter/fileexporter v0.87.0
2023-10-23T12:19:11.886+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/exporter/googlecloudexporter v0.87.0
2023-10-23T12:19:11.889+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/exporter/googlecloudpubsubexporter v0.87.0
2023-10-23T12:19:11.910+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/exporter/googlemanagedprometheusexporter v0.87.0
2023-10-23T12:19:11.915+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/exporter/influxdbexporter v0.87.0
2023-10-23T12:19:11.922+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/exporter/instanaexporter v0.87.0
2023-10-23T12:19:11.928+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/exporter/kafkaexporter v0.87.0
2023-10-23T12:19:11.928+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/exporter/loadbalancingexporter v0.87.0
2023-10-23T12:19:11.932+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/exporter/logicmonitorexporter v0.87.0
2023-10-23T12:19:11.939+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/exporter/logzioexporter v0.87.0
2023-10-23T12:19:11.940+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/exporter/lokiexporter v0.87.0
2023-10-23T12:19:11.941+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/exporter/mezmoexporter v0.87.0
2023-10-23T12:19:11.949+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/exporter/opencensusexporter v0.87.0
2023-10-23T12:19:11.949+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/exporter/prometheusexporter v0.87.0
2023-10-23T12:19:11.951+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/exporter/prometheusremotewriteexporter v0.87.0
2023-10-23T12:19:11.954+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/exporter/pulsarexporter v0.87.0
2023-10-23T12:19:11.957+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/exporter/sapmexporter v0.87.0
2023-10-23T12:19:11.958+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/exporter/sentryexporter v0.87.0
2023-10-23T12:19:11.958+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/exporter/signalfxexporter v0.87.0
2023-10-23T12:19:11.960+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/exporter/skywalkingexporter v0.87.0
2023-10-23T12:19:11.961+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/exporter/splunkhecexporter v0.87.0
2023-10-23T12:19:11.962+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/exporter/sumologicexporter v0.87.0
2023-10-23T12:19:11.965+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/exporter/tanzuobservabilityexporter v0.87.0
2023-10-23T12:19:11.973+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/exporter/tencentcloudlogserviceexporter v0.87.0
2023-10-23T12:19:11.979+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/exporter/zipkinexporter v0.87.0
2023-10-23T12:19:11.983+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/extension/asapauthextension v0.87.0
2023-10-23T12:19:11.985+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/extension/awsproxy v0.87.0
2023-10-23T12:19:11.985+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/extension/basicauthextension v0.87.0
2023-10-23T12:19:11.988+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/extension/bearertokenauthextension v0.87.0
2023-10-23T12:19:11.991+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/extension/headerssetterextension v0.87.0
2023-10-23T12:19:11.996+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/extension/healthcheckextension v0.87.0
2023-10-23T12:19:11.998+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/extension/httpforwarder v0.87.0
2023-10-23T12:19:12.002+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/extension/jaegerremotesampling v0.87.0
2023-10-23T12:19:12.007+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/extension/oauth2clientauthextension v0.87.0
2023-10-23T12:19:12.013+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/extension/observer/dockerobserver v0.87.0
2023-10-23T12:19:12.013+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/extension/observer/ecsobserver v0.87.0
2023-10-23T12:19:12.016+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/extension/observer/ecstaskobserver v0.87.0
2023-10-23T12:19:12.017+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/extension/observer/k8sobserver v0.87.0
2023-10-23T12:19:12.018+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/extension/observer/hostobserver v0.87.0
2023-10-23T12:19:12.020+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/extension/oidcauthextension v0.87.0
2023-10-23T12:19:12.022+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/extension/pprofextension v0.87.0
2023-10-23T12:19:12.022+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/extension/sigv4authextension v0.87.0
2023-10-23T12:19:12.022+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/extension/storage v0.87.0
2023-10-23T12:19:12.028+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/processor/attributesprocessor v0.87.0
2023-10-23T12:19:12.029+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/processor/cumulativetodeltaprocessor v0.87.0
2023-10-23T12:19:12.032+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/processor/datadogprocessor v0.87.0
2023-10-23T12:19:12.034+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/processor/deltatorateprocessor v0.87.0
2023-10-23T12:19:12.035+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/processor/filterprocessor v0.87.0
2023-10-23T12:19:12.043+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/processor/groupbyattrsprocessor v0.87.0
2023-10-23T12:19:12.043+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/processor/groupbytraceprocessor v0.87.0
2023-10-23T12:19:12.045+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/processor/k8sattributesprocessor v0.87.0
2023-10-23T12:19:12.054+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/processor/metricsgenerationprocessor v0.87.0
2023-10-23T12:19:12.058+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/processor/metricstransformprocessor v0.87.0
2023-10-23T12:19:12.059+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/processor/probabilisticsamplerprocessor v0.87.0
2023-10-23T12:19:12.063+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/processor/redactionprocessor v0.87.0
2023-10-23T12:19:12.065+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/processor/remoteobserverprocessor v0.87.0
2023-10-23T12:19:12.067+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/processor/resourcedetectionprocessor v0.87.0
2023-10-23T12:19:12.068+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/processor/resourceprocessor v0.87.0
2023-10-23T12:19:12.078+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/processor/routingprocessor v0.87.0
2023-10-23T12:19:12.084+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/processor/servicegraphprocessor v0.87.0
2023-10-23T12:19:12.095+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/processor/spanmetricsprocessor v0.87.0
2023-10-23T12:19:12.096+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/processor/spanprocessor v0.87.0
2023-10-23T12:19:12.099+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/processor/tailsamplingprocessor v0.87.0
2023-10-23T12:19:12.102+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/processor/transformprocessor v0.87.0
2023-10-23T12:19:12.105+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/activedirectorydsreceiver v0.87.0
2023-10-23T12:19:12.109+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/aerospikereceiver v0.87.0
2023-10-23T12:19:12.114+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/apachereceiver v0.87.0
2023-10-23T12:19:12.117+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/apachesparkreceiver v0.87.0
2023-10-23T12:19:12.117+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/awscloudwatchreceiver v0.87.0
2023-10-23T12:19:12.118+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/awscontainerinsightreceiver v0.87.0
2023-10-23T12:19:12.118+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/awsfirehosereceiver v0.87.0
2023-10-23T12:19:12.118+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/awsxrayreceiver v0.87.0
2023-10-23T12:19:12.118+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/awsecscontainermetricsreceiver v0.87.0
2023-10-23T12:19:12.123+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/azureblobreceiver v0.87.0
2023-10-23T12:19:12.128+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/azureeventhubreceiver v0.87.0
2023-10-23T12:19:12.138+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/azuremonitorreceiver v0.87.0
2023-10-23T12:19:12.146+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/bigipreceiver v0.87.0
2023-10-23T12:19:12.147+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/carbonreceiver v0.87.0
2023-10-23T12:19:12.148+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/chronyreceiver v0.87.0
2023-10-23T12:19:12.150+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/cloudflarereceiver v0.87.0
2023-10-23T12:19:12.156+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/cloudfoundryreceiver v0.87.0
2023-10-23T12:19:12.162+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/collectdreceiver v0.87.0
2023-10-23T12:19:12.194+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/couchdbreceiver v0.87.0
2023-10-23T12:19:12.197+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/datadogreceiver v0.87.0
2023-10-23T12:19:12.203+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/dockerstatsreceiver v0.87.0
2023-10-23T12:19:12.206+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/expvarreceiver v0.87.0
2023-10-23T12:19:12.206+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/elasticsearchreceiver v0.87.0
2023-10-23T12:19:12.210+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/filelogreceiver v0.87.0
2023-10-23T12:19:12.210+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/filereceiver v0.87.0
2023-10-23T12:19:12.211+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/filestatsreceiver v0.87.0
2023-10-23T12:19:12.214+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/flinkmetricsreceiver v0.87.0
2023-10-23T12:19:12.215+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/fluentforwardreceiver v0.87.0
2023-10-23T12:19:12.216+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/googlecloudpubsubreceiver v0.87.0
2023-10-23T12:19:12.217+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/googlecloudspannerreceiver v0.87.0
2023-10-23T12:19:12.221+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/haproxyreceiver v0.87.0
2023-10-23T12:19:12.225+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/hostmetricsreceiver v0.87.0
2023-10-23T12:19:12.233+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/httpcheckreceiver v0.87.0
2023-10-23T12:19:12.234+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/iisreceiver v0.87.0
2023-10-23T12:19:12.237+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/influxdbreceiver v0.87.0
2023-10-23T12:19:12.242+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/jaegerreceiver v0.87.0
2023-10-23T12:19:12.243+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/jmxreceiver v0.87.0
2023-10-23T12:19:12.243+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/journaldreceiver v0.87.0
2023-10-23T12:19:12.250+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/k8sclusterreceiver v0.87.0
2023-10-23T12:19:12.250+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/k8seventsreceiver v0.87.0
2023-10-23T12:19:12.252+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/k8sobjectsreceiver v0.87.0
2023-10-23T12:19:12.253+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/kafkametricsreceiver v0.87.0
2023-10-23T12:19:12.255+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/kafkareceiver v0.87.0
2023-10-23T12:19:12.266+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/kubeletstatsreceiver v0.87.0
2023-10-23T12:19:12.270+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/lokireceiver v0.87.0
2023-10-23T12:19:12.274+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/memcachedreceiver v0.87.0
2023-10-23T12:19:12.275+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/mongodbatlasreceiver v0.87.0
2023-10-23T12:19:12.277+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/mongodbreceiver v0.87.0
2023-10-23T12:19:12.284+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/mysqlreceiver v0.87.0
2023-10-23T12:19:12.295+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/nginxreceiver v0.87.0
2023-10-23T12:19:12.299+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/nsxtreceiver v0.87.0
2023-10-23T12:19:12.301+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/opencensusreceiver v0.87.0
2023-10-23T12:19:12.305+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/oracledbreceiver v0.87.0
2023-10-23T12:19:12.306+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/otlpjsonfilereceiver v0.87.0
2023-10-23T12:19:12.307+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/podmanreceiver v0.87.0
2023-10-23T12:19:12.307+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/prometheusreceiver v0.87.0
2023-10-23T12:19:12.308+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/postgresqlreceiver v0.87.0
2023-10-23T12:19:12.311+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/pulsarreceiver v0.87.0
2023-10-23T12:19:12.313+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/purefareceiver v0.87.0
2023-10-23T12:19:12.319+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/purefbreceiver v0.87.0
2023-10-23T12:19:12.329+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/rabbitmqreceiver v0.87.0
2023-10-23T12:19:12.337+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/receivercreator v0.87.0
2023-10-23T12:19:12.343+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/redisreceiver v0.87.0
2023-10-23T12:19:12.347+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/riakreceiver v0.87.0
2023-10-23T12:19:12.347+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/sapmreceiver v0.87.0
2023-10-23T12:19:12.352+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/signalfxreceiver v0.87.0
2023-10-23T12:19:12.354+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/simpleprometheusreceiver v0.87.0
2023-10-23T12:19:12.356+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/skywalkingreceiver v0.87.0
2023-10-23T12:19:12.359+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/snmpreceiver v0.87.0
2023-10-23T12:19:12.359+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/snowflakereceiver v0.87.0
2023-10-23T12:19:12.360+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/solacereceiver v0.87.0
2023-10-23T12:19:12.361+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/splunkhecreceiver v0.87.0
2023-10-23T12:19:12.361+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/sqlqueryreceiver v0.87.0
2023-10-23T12:19:12.365+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/sqlserverreceiver v0.87.0
2023-10-23T12:19:12.366+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/sshcheckreceiver v0.87.0
2023-10-23T12:19:12.368+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/statsdreceiver v0.87.0
2023-10-23T12:19:12.371+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/syslogreceiver v0.87.0
2023-10-23T12:19:12.380+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/tcplogreceiver v0.87.0
2023-10-23T12:19:12.404+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/udplogreceiver v0.87.0
2023-10-23T12:19:12.405+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/vcenterreceiver v0.87.0
2023-10-23T12:19:12.410+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/wavefrontreceiver v0.87.0
2023-10-23T12:19:12.411+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/webhookeventreceiver v0.87.0
2023-10-23T12:19:12.417+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/windowseventlogreceiver v0.87.0
2023-10-23T12:19:12.419+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/windowsperfcountersreceiver v0.87.0
2023-10-23T12:19:12.419+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/zipkinreceiver v0.87.0
2023-10-23T12:19:12.422+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/receiver/zookeeperreceiver v0.87.0
2023-10-23T12:19:12.424+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/collector v0.87.0
2023-10-23T12:19:12.430+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/collector/connector/forwardconnector v0.87.0
2023-10-23T12:19:12.433+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/collector/exporter/debugexporter v0.87.0
2023-10-23T12:19:12.435+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/collector/exporter/loggingexporter v0.87.0
2023-10-23T12:19:12.439+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/collector/exporter/otlpexporter v0.87.0
2023-10-23T12:19:12.440+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/collector/exporter/otlphttpexporter v0.87.0
2023-10-23T12:19:12.442+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/collector/extension/ballastextension v0.87.0
2023-10-23T12:19:12.443+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/collector/extension/zpagesextension v0.87.0
2023-10-23T12:19:12.450+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/collector/processor/batchprocessor v0.87.0
2023-10-23T12:19:12.453+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/collector/processor/memorylimiterprocessor v0.87.0
2023-10-23T12:19:12.458+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/collector/receiver/otlpreceiver v0.87.0
2023-10-23T12:19:12.711+0200    INFO    zapio/writer.go:146     go: downloading golang.org/x/sys v0.13.0
2023-10-23T12:19:12.711+0200    INFO    zapio/writer.go:146     go: downloading github.com/stretchr/testify v1.8.4
2023-10-23T12:19:12.711+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/collector/component v0.87.0
2023-10-23T12:19:12.711+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/collector/connector v0.87.0
2023-10-23T12:19:12.711+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/collector/exporter v0.87.0
2023-10-23T12:19:12.711+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/collector/extension v0.87.0
2023-10-23T12:19:12.711+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/collector/otelcol v0.87.0
2023-10-23T12:19:12.711+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/collector/processor v0.87.0
2023-10-23T12:19:12.712+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/collector/receiver v0.87.0
2023-10-23T12:19:12.712+0200    INFO    zapio/writer.go:146     go: downloading github.com/cenkalti/backoff/v4 v4.2.1
2023-10-23T12:19:12.712+0200    INFO    zapio/writer.go:146     go: downloading github.com/scalyr/dataset-go v0.14.0
2023-10-23T12:19:12.712+0200    INFO    zapio/writer.go:146     go: downloading github.com/google/uuid v1.3.1
2023-10-23T12:19:12.712+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/collector/config/configopaque v0.87.0
2023-10-23T12:19:12.712+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/collector/confmap v0.87.0
2023-10-23T12:19:12.753+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/collector/pdata v1.0.0-rcv0016
2023-10-23T12:19:12.753+0200    INFO    zapio/writer.go:146     go: downloading github.com/cenkalti/backoff v2.2.1+incompatible
2023-10-23T12:19:12.761+0200    INFO    zapio/writer.go:146     go: downloading go.uber.org/zap v1.26.0
2023-10-23T12:19:12.761+0200    INFO    zapio/writer.go:146     go: downloading golang.org/x/time v0.3.0
2023-10-23T12:19:12.763+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/internal/coreinternal v0.87.0
2023-10-23T12:19:12.776+0200    INFO    zapio/writer.go:146     go: downloading github.com/Azure/azure-kusto-go v0.13.1
2023-10-23T12:19:12.777+0200    INFO    zapio/writer.go:146     go: downloading github.com/gocql/gocql v1.3.1
2023-10-23T12:19:12.777+0200    INFO    zapio/writer.go:146     go: downloading github.com/json-iterator/go v1.1.12
2023-10-23T12:19:12.782+0200    INFO    zapio/writer.go:146     go: downloading github.com/dynatrace-oss/dynatrace-metric-utils-go v0.5.0
2023-10-23T12:19:12.795+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/internal/common v0.87.0
2023-10-23T12:19:12.796+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/pkg/resourcetotelemetry v0.87.0
2023-10-23T12:19:12.818+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/collector/config/confighttp v0.87.0
2023-10-23T12:19:12.818+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/collector/consumer v0.87.0
2023-10-23T12:19:12.844+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/collector/config/configtls v0.87.0
2023-10-23T12:19:12.858+0200    INFO    zapio/writer.go:146     go: downloading github.com/DataDog/datadog-agent/pkg/proto v0.48.0-beta.1
2023-10-23T12:19:12.858+0200    INFO    zapio/writer.go:146     go: downloading github.com/DataDog/opentelemetry-mapping-go/pkg/otlp/metrics v0.8.0
2023-10-23T12:19:12.859+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/internal/datadog v0.87.0
2023-10-23T12:19:12.862+0200    INFO    zapio/writer.go:146     go: downloading github.com/aws/aws-sdk-go v1.45.24
2023-10-23T12:19:12.876+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/internal/aws/awsutil v0.87.0
2023-10-23T12:19:12.879+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/internal/aws/cwlogs v0.87.0
2023-10-23T12:19:12.880+0200    INFO    zapio/writer.go:146     go: downloading go.uber.org/multierr v1.11.0
2023-10-23T12:19:12.889+0200    INFO    zapio/writer.go:146     go: downloading github.com/aws/aws-sdk-go-v2 v1.21.1
2023-10-23T12:19:12.889+0200    INFO    zapio/writer.go:146     go: downloading github.com/aws/aws-sdk-go-v2/config v1.18.44
2023-10-23T12:19:12.896+0200    INFO    zapio/writer.go:146     go: downloading github.com/aws/aws-sdk-go-v2/credentials v1.13.42
2023-10-23T12:19:12.897+0200    INFO    zapio/writer.go:146     go: downloading github.com/aws/aws-sdk-go-v2/service/kinesis v1.19.1
2023-10-23T12:19:12.909+0200    INFO    zapio/writer.go:146     go: downloading github.com/aws/aws-sdk-go-v2/service/sts v1.23.1
2023-10-23T12:19:12.909+0200    INFO    zapio/writer.go:146     go: downloading github.com/lightstep/go-expohisto v1.0.0
2023-10-23T12:19:12.909+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/collector/semconv v0.87.0
2023-10-23T12:19:12.914+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/pkg/pdatautil v0.87.0
2023-10-23T12:19:12.916+0200    INFO    zapio/writer.go:146     go: downloading github.com/tilinna/clock v1.1.0
2023-10-23T12:19:12.930+0200    INFO    zapio/writer.go:146     go: downloading google.golang.org/grpc v1.58.2
2023-10-23T12:19:12.930+0200    INFO    zapio/writer.go:146     go: downloading github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/collector v0.44.0
2023-10-23T12:19:12.932+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/collector/featuregate v1.0.0-rcv0016
2023-10-23T12:19:12.933+0200    INFO    zapio/writer.go:146     go: downloading github.com/aliyun/aliyun-log-go-sdk v0.1.60
2023-10-23T12:19:12.951+0200    INFO    zapio/writer.go:146     go: downloading github.com/gogo/protobuf v1.3.2
2023-10-23T12:19:12.979+0200    INFO    zapio/writer.go:146     go: downloading golang.org/x/oauth2 v0.13.0
2023-10-23T12:19:12.997+0200    INFO    zapio/writer.go:146     go: downloading google.golang.org/api v0.146.0
2023-10-23T12:19:12.997+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/collector/config/configcompression v0.87.0
2023-10-23T12:19:13.022+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/collector/config/configgrpc v0.87.0
2023-10-23T12:19:13.022+0200    INFO    zapio/writer.go:146     go: downloading google.golang.org/genproto/googleapis/rpc v0.0.0-20230920204549-e6e6cdab5c13
2023-10-23T12:19:13.034+0200    INFO    zapio/writer.go:146     go: downloading github.com/klauspost/compress v1.17.0
2023-10-23T12:19:13.035+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/internal/sharedcomponent v0.87.0
2023-10-23T12:19:13.035+0200    INFO    zapio/writer.go:146     go: downloading gopkg.in/natefinch/lumberjack.v2 v2.2.1
2023-10-23T12:19:13.036+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/pkg/ottl v0.87.0
2023-10-23T12:19:13.046+0200    INFO    zapio/writer.go:146     go: downloading google.golang.org/genproto v0.0.0-20230913181813-007df8e322eb
2023-10-23T12:19:13.054+0200    INFO    zapio/writer.go:146     go: downloading cloud.google.com/go/pubsub v1.33.0
2023-10-23T12:19:13.126+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/internal/filter v0.87.0
2023-10-23T12:19:13.152+0200    INFO    zapio/writer.go:146     go: downloading cloud.google.com/go v0.110.2
2023-10-23T12:19:13.229+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/pkg/pdatatest v0.87.0
2023-10-23T12:19:13.270+0200    INFO    zapio/writer.go:146     go: downloading github.com/elastic/go-elasticsearch/v7 v7.17.10
2023-10-23T12:19:13.506+0200    INFO    zapio/writer.go:146     go: downloading golang.org/x/exp v0.0.0-20230713183714-613f0c0eb8a1
2023-10-23T12:19:13.506+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/internal/aws/metrics v0.87.0
2023-10-23T12:19:13.555+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/internal/aws/xray v0.87.0
2023-10-23T12:19:13.584+0200    INFO    zapio/writer.go:146     go: downloading github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/collector/googlemanagedprometheus v0.44.0
2023-10-23T12:19:13.585+0200    INFO    zapio/writer.go:146     go: downloading github.com/microsoft/ApplicationInsights-Go v0.4.4
2023-10-23T12:19:13.620+0200    INFO    zapio/writer.go:146     go: downloading golang.org/x/net v0.17.0
2023-10-23T12:19:13.639+0200    INFO    zapio/writer.go:146     go: downloading github.com/influxdata/influxdb-observability/common v0.5.8
2023-10-23T12:19:13.639+0200    INFO    zapio/writer.go:146     go: downloading github.com/influxdata/influxdb-observability/otel2influx v0.5.8
2023-10-23T12:19:13.662+0200    INFO    zapio/writer.go:146     go: downloading github.com/influxdata/line-protocol/v2 v2.2.1
2023-10-23T12:19:13.670+0200    INFO    zapio/writer.go:146     go: downloading github.com/ClickHouse/clickhouse-go/v2 v2.14.2
2023-10-23T12:19:13.672+0200    INFO    zapio/writer.go:146     go: downloading github.com/hashicorp/go-hclog v1.5.0
2023-10-23T12:19:13.678+0200    INFO    zapio/writer.go:146     go: downloading github.com/jaegertracing/jaeger v1.48.0
2023-10-23T12:19:13.817+0200    INFO    zapio/writer.go:146     go: downloading github.com/influxdata/line-protocol v0.0.0-20200327222509-2487e7298839
2023-10-23T12:19:13.914+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/jaeger v0.87.0
2023-10-23T12:19:14.076+0200    INFO    zapio/writer.go:146     go: downloading google.golang.org/protobuf v1.31.0
2023-10-23T12:19:14.276+0200    INFO    zapio/writer.go:146     go: downloading github.com/IBM/sarama v1.41.2
2023-10-23T12:19:14.276+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/internal/kafka v0.87.0
2023-10-23T12:19:14.276+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/zipkin v0.87.0
2023-10-23T12:19:14.333+0200    INFO    zapio/writer.go:146     go: downloading github.com/openzipkin/zipkin-go v0.4.2
2023-10-23T12:19:14.407+0200    INFO    zapio/writer.go:146     go: downloading github.com/logicmonitor/lm-data-sdk-go v1.0.0
2023-10-23T12:19:14.456+0200    INFO    zapio/writer.go:146     go: downloading github.com/golang/snappy v0.0.4
2023-10-23T12:19:14.456+0200    INFO    zapio/writer.go:146     go: downloading go.opencensus.io v0.24.0
2023-10-23T12:19:14.456+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/loki v0.87.0
2023-10-23T12:19:14.489+0200    INFO    zapio/writer.go:146     go: downloading github.com/grafana/loki/pkg/push v0.0.0-20230321110627-5c3d204ebf5d
2023-10-23T12:19:14.510+0200    INFO    zapio/writer.go:146     go: downloading github.com/prometheus/common v0.44.0
2023-10-23T12:19:14.532+0200    INFO    zapio/writer.go:146     go: downloading github.com/census-instrumentation/opencensus-proto v0.4.1
2023-10-23T12:19:14.611+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/opencensus v0.87.0
2023-10-23T12:19:14.611+0200    INFO    zapio/writer.go:146     go: downloading github.com/getsentry/sentry-go v0.25.0
2023-10-23T12:19:14.611+0200    INFO    zapio/writer.go:146     go: downloading github.com/google/go-cmp v0.5.9
2023-10-23T12:19:14.675+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/internal/splunk v0.87.0
2023-10-23T12:19:14.675+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/pkg/batchperresourceattr v0.87.0
2023-10-23T12:19:14.795+0200    INFO    zapio/writer.go:146     go: downloading github.com/signalfx/sapm-proto v0.13.0
2023-10-23T12:19:14.802+0200    INFO    zapio/writer.go:146     go: downloading github.com/apache/pulsar-client-go v0.8.1
2023-10-23T12:19:14.819+0200    INFO    zapio/writer.go:146     go: downloading github.com/tencentcloud/tencentcloud-sdk-go/tencentcloud/common v1.0.762
2023-10-23T12:19:14.819+0200    INFO    zapio/writer.go:146     go: downloading github.com/pierrec/lz4 v2.6.1+incompatible
2023-10-23T12:19:14.863+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/prometheus v0.87.0
2023-10-23T12:19:14.892+0200    INFO    zapio/writer.go:146     go: downloading github.com/prometheus/client_golang v1.17.0
2023-10-23T12:19:14.933+0200    INFO    zapio/writer.go:146     go: downloading github.com/prometheus/client_model v0.5.0
2023-10-23T12:19:14.933+0200    INFO    zapio/writer.go:146     go: downloading github.com/prometheus/prometheus v0.47.1
2023-10-23T12:19:14.971+0200    INFO    zapio/writer.go:146     go: downloading gopkg.in/yaml.v2 v2.4.0
2023-10-23T12:19:14.989+0200    INFO    zapio/writer.go:146     go: downloading github.com/tg123/go-htpasswd v1.2.1
2023-10-23T12:19:14.995+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/collector/extension/auth v0.87.0
2023-10-23T12:19:15.012+0200    INFO    zapio/writer.go:146     go: downloading skywalking.apache.org/repo/goapi v0.0.0-20220121092418-9c455d0dda3f
2023-10-23T12:19:15.584+0200    INFO    zapio/writer.go:146     go: downloading bitbucket.org/atlassian/go-asap/v2 v2.6.0
2023-10-23T12:19:15.584+0200    INFO    zapio/writer.go:146     go: downloading github.com/SermoDigital/jose v0.9.2-0.20161205224733-f6df55f235c2
2023-10-23T12:19:15.584+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/pkg/batchpersignal v0.87.0
2023-10-23T12:19:15.662+0200    INFO    zapio/writer.go:146     go: downloading k8s.io/api v0.28.2
2023-10-23T12:19:15.662+0200    INFO    zapio/writer.go:146     go: downloading k8s.io/apimachinery v0.28.2
2023-10-23T12:19:15.662+0200    INFO    zapio/writer.go:146     go: downloading k8s.io/client-go v0.28.2
2023-10-23T12:19:16.217+0200    INFO    zapio/writer.go:146     go: downloading k8s.io/utils v0.0.0-20230711102312-30195339c3c7
2023-10-23T12:19:16.268+0200    INFO    zapio/writer.go:146     go: downloading sigs.k8s.io/controller-runtime v0.16.2
2023-10-23T12:19:16.575+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/internal/aws/proxy v0.87.0
2023-10-23T12:19:16.575+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/collector/config/confignet v0.87.0
2023-10-23T12:19:16.616+0200    INFO    zapio/writer.go:146     go: downloading github.com/fsnotify/fsnotify v1.6.0
2023-10-23T12:19:16.624+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/prometheusremotewrite v0.87.0
2023-10-23T12:19:16.643+0200    INFO    zapio/writer.go:146     go: downloading github.com/tidwall/wal v1.1.7
2023-10-23T12:19:16.650+0200    INFO    zapio/writer.go:146     go: downloading github.com/DataDog/datadog-agent/pkg/trace v0.48.0-beta.1
2023-10-23T12:19:16.787+0200    INFO    zapio/writer.go:146     go: downloading github.com/DataDog/datadog-api-client-go/v2 v2.17.0
2023-10-23T12:19:16.787+0200    INFO    zapio/writer.go:146     go: downloading github.com/DataDog/opentelemetry-mapping-go/pkg/inframetadata v0.8.0
2023-10-23T12:19:16.802+0200    INFO    zapio/writer.go:146     go: downloading github.com/DataDog/opentelemetry-mapping-go/pkg/otlp/attributes v0.8.0
2023-10-23T12:19:16.822+0200    INFO    zapio/writer.go:146     go: downloading github.com/DataDog/opentelemetry-mapping-go/pkg/otlp/logs v0.8.0
2023-10-23T12:19:16.838+0200    INFO    zapio/writer.go:146     go: downloading gopkg.in/zorkian/go-datadog-api.v2 v2.30.0
2023-10-23T12:19:16.920+0200    INFO    zapio/writer.go:146     go: downloading github.com/DataDog/agent-payload/v5 v5.0.89
2023-10-23T12:19:17.049+0200    INFO    zapio/writer.go:146     go: downloading github.com/wavefronthq/wavefront-sdk-go v0.14.0
2023-10-23T12:19:17.096+0200    INFO    zapio/writer.go:146     go: downloading github.com/docker/docker v24.0.6+incompatible
2023-10-23T12:19:17.096+0200    INFO    zapio/writer.go:146     go: downloading github.com/docker/go-connections v0.4.0
2023-10-23T12:19:17.116+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/extension/observer v0.87.0
2023-10-23T12:19:17.131+0200    INFO    zapio/writer.go:146     go: downloading github.com/testcontainers/testcontainers-go v0.25.0
2023-10-23T12:19:17.132+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/internal/docker v0.87.0
2023-10-23T12:19:17.246+0200    INFO    zapio/writer.go:146     go: downloading github.com/coreos/go-oidc v2.2.1+incompatible
2023-10-23T12:19:17.281+0200    INFO    zapio/writer.go:146     go: downloading github.com/shirou/gopsutil/v3 v3.23.9
2023-10-23T12:19:17.514+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/internal/aws/ecsutil v0.87.0
2023-10-23T12:19:17.528+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/internal/k8sconfig v0.87.0
2023-10-23T12:19:17.528+0200    INFO    zapio/writer.go:146     go: downloading github.com/DataDog/sketches-go v1.4.3
2023-10-23T12:19:17.529+0200    INFO    zapio/writer.go:146     go: downloading github.com/jackc/pgx/v4 v4.18.1
2023-10-23T12:19:17.530+0200    INFO    zapio/writer.go:146     go: downloading github.com/mattn/go-sqlite3 v1.14.17
2023-10-23T12:19:17.542+0200    INFO    zapio/writer.go:146     go: downloading go.etcd.io/bbolt v1.3.7
2023-10-23T12:19:17.587+0200    INFO    zapio/writer.go:146     go: downloading github.com/hashicorp/golang-lru v1.0.2
2023-10-23T12:19:17.604+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/pkg/experimentalmetricmetadata v0.87.0
2023-10-23T12:19:17.615+0200    INFO    zapio/writer.go:146     go: downloading github.com/signalfx/com_signalfx_metrics_protobuf v0.0.3
2023-10-23T12:19:17.620+0200    INFO    zapio/writer.go:146     go: downloading gopkg.in/yaml.v3 v3.0.1
2023-10-23T12:19:17.634+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/collector/config/configtelemetry v0.87.0
2023-10-23T12:19:17.668+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/otel v1.19.0
2023-10-23T12:19:17.691+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/otel/metric v1.19.0
2023-10-23T12:19:17.856+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/otel/trace v1.19.0
2023-10-23T12:19:17.856+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/internal/k8stest v0.87.0
2023-10-23T12:19:17.917+0200    INFO    zapio/writer.go:146     go: downloading go.uber.org/goleak v1.2.1
2023-10-23T12:19:17.917+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/pkg/winperfcounters v0.87.0
2023-10-23T12:19:17.917+0200    INFO    zapio/writer.go:146     go: downloading github.com/aerospike/aerospike-client-go/v6 v6.13.0
2023-10-23T12:19:17.917+0200    INFO    zapio/writer.go:146     go: downloading github.com/Azure/azure-event-hubs-go/v3 v3.6.1
2023-10-23T12:19:17.935+0200    INFO    zapio/writer.go:146     go: downloading github.com/Azure/azure-sdk-for-go/sdk/storage/azblob v1.1.0
2023-10-23T12:19:17.985+0200    INFO    zapio/writer.go:146     go: downloading github.com/Azure/azure-sdk-for-go/sdk/azcore v1.7.0
2023-10-23T12:19:18.037+0200    INFO    zapio/writer.go:146     go: downloading github.com/Azure/azure-sdk-for-go v68.0.0+incompatible
2023-10-23T12:19:18.183+0200    INFO    zapio/writer.go:146     go: downloading github.com/Azure/azure-sdk-for-go/sdk/azidentity v1.3.0
2023-10-23T12:19:18.218+0200    INFO    zapio/writer.go:146     go: downloading github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/monitor/armmonitor v0.8.0
2023-10-23T12:19:18.444+0200    INFO    zapio/writer.go:146     go: downloading github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/resources/armresources v1.0.0
2023-10-23T12:19:19.721+0200    INFO    zapio/writer.go:146     go: downloading github.com/Azure/azure-amqp-common-go/v4 v4.2.0
2023-10-23T12:19:19.740+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/pkg/stanza v0.87.0
2023-10-23T12:19:19.991+0200    INFO    zapio/writer.go:146     go: downloading github.com/relvacode/iso8601 v1.3.0
2023-10-23T12:19:20.032+0200    INFO    zapio/writer.go:146     go: downloading code.cloudfoundry.org/go-loggregator v7.4.0+incompatible
2023-10-23T12:19:20.099+0200    INFO    zapio/writer.go:146     go: downloading github.com/cloudfoundry-incubator/uaago v0.0.0-20190307164349-8136b7bbe76e
2023-10-23T12:19:20.152+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/otel/sdk v1.19.0
2023-10-23T12:19:20.216+0200    INFO    zapio/writer.go:146     go: downloading github.com/vmihailenco/msgpack/v4 v4.3.12
2023-10-23T12:19:20.287+0200    INFO    zapio/writer.go:146     go: downloading github.com/observiq/nanojack v0.0.0-20201106172433-343928847ebc
2023-10-23T12:19:20.327+0200    INFO    zapio/writer.go:146     go: downloading github.com/bmatcuk/doublestar/v4 v4.6.0
2023-10-23T12:19:20.358+0200    INFO    zapio/writer.go:146     go: downloading github.com/tinylib/msgp v1.1.8
2023-10-23T12:19:20.419+0200    INFO    zapio/writer.go:146     go: downloading github.com/influxdata/influxdb-observability/influx2otel v0.5.8
2023-10-23T12:19:20.464+0200    INFO    zapio/writer.go:146     go: downloading github.com/influxdata/influxdb-client-go/v2 v2.12.3
2023-10-23T12:19:20.595+0200    INFO    zapio/writer.go:146     go: downloading github.com/influxdata/influxdb1-client v0.0.0-20220302092344-a9ab5670611c
2023-10-23T12:19:20.629+0200    INFO    zapio/writer.go:146     go: downloading github.com/hashicorp/go-version v1.6.0
2023-10-23T12:19:20.677+0200    INFO    zapio/writer.go:146     go: downloading github.com/apache/thrift v0.19.0
2023-10-23T12:19:21.984+0200    INFO    zapio/writer.go:146     go: downloading github.com/grobie/gomemcache v0.0.0-20180201122607-1f779c573665
2023-10-23T12:19:22.027+0200    INFO    zapio/writer.go:146     go: downloading github.com/gorilla/mux v1.8.0
2023-10-23T12:19:22.049+0200    INFO    zapio/writer.go:146     go: downloading github.com/nginxinc/nginx-prometheus-exporter v0.8.1-0.20201110005315-f5a5f8086c19
2023-10-23T12:19:22.213+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/internal/aws/containerinsight v0.87.0
2023-10-23T12:19:22.269+0200    INFO    zapio/writer.go:146     go: downloading golang.org/x/crypto v0.14.0
2023-10-23T12:19:22.276+0200    INFO    zapio/writer.go:146     go: downloading github.com/grpc-ecosystem/grpc-gateway/v2 v2.18.0
2023-10-23T12:19:22.276+0200    INFO    zapio/writer.go:146     go: downloading github.com/go-sql-driver/mysql v1.7.1
2023-10-23T12:19:22.276+0200    INFO    zapio/writer.go:146     go: downloading github.com/soheilhy/cmux v0.1.5
2023-10-23T12:19:22.276+0200    INFO    zapio/writer.go:146     go: downloading github.com/rs/cors v1.10.1
2023-10-23T12:19:22.276+0200    INFO    zapio/writer.go:146     go: downloading github.com/sijms/go-ora/v2 v2.7.19
2023-10-23T12:19:22.276+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/internal/kubelet v0.87.0
2023-10-23T12:19:22.277+0200    INFO    zapio/writer.go:146     go: downloading go.mongodb.org/mongo-driver v1.12.1
2023-10-23T12:19:22.512+0200    INFO    zapio/writer.go:146     go: downloading github.com/lib/pq v1.10.9
2023-10-23T12:19:22.512+0200    INFO    zapio/writer.go:146     go: downloading github.com/openshift/client-go v0.0.0-20210521082421-73d9475a9142
2023-10-23T12:19:22.550+0200    INFO    zapio/writer.go:146     go: downloading github.com/openshift/api v0.0.0-20230726162818-81f778f3b3ec
2023-10-23T12:19:22.609+0200    INFO    zapio/writer.go:146     go: downloading go.mongodb.org/atlas v0.33.0
2023-10-23T12:19:22.724+0200    INFO    zapio/writer.go:146     go: downloading github.com/antonmedv/expr v1.15.3
2023-10-23T12:19:23.226+0200    INFO    zapio/writer.go:146     go: downloading github.com/spf13/cast v1.5.1
2023-10-23T12:19:23.226+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/signalfx v0.87.0
2023-10-23T12:19:23.229+0200    INFO    zapio/writer.go:146     go: downloading github.com/go-redis/redis/v7 v7.4.1
2023-10-23T12:19:23.231+0200    INFO    zapio/writer.go:146     go: downloading github.com/snowflakedb/gosnowflake v1.6.25
2023-10-23T12:19:23.231+0200    INFO    zapio/writer.go:146     go: downloading github.com/DATA-DOG/go-sqlmock v1.5.0
2023-10-23T12:19:23.265+0200    INFO    zapio/writer.go:146     go: downloading github.com/julienschmidt/httprouter v1.3.0
2023-10-23T12:19:23.298+0200    INFO    zapio/writer.go:146     go: downloading github.com/pkg/sftp v1.13.5
2023-10-23T12:19:23.298+0200    INFO    zapio/writer.go:146     go: downloading github.com/SAP/go-hdb v1.3.10
2023-10-23T12:19:23.299+0200    INFO    zapio/writer.go:146     go: downloading github.com/denisenkom/go-mssqldb v0.12.2
2023-10-23T12:19:23.308+0200    INFO    zapio/writer.go:146     go: downloading github.com/Azure/go-amqp v1.0.2
2023-10-23T12:19:23.324+0200    INFO    zapio/writer.go:146     go: downloading github.com/gosnmp/gosnmp v1.35.0
2023-10-23T12:19:23.371+0200    INFO    zapio/writer.go:146     go: downloading github.com/go-kit/log v0.2.1
2023-10-23T12:19:23.414+0200    INFO    zapio/writer.go:146     go: downloading github.com/mitchellh/hashstructure/v2 v2.0.2
2023-10-23T12:19:23.422+0200    INFO    zapio/writer.go:146     go: downloading github.com/vmware/govmomi v0.32.0
2023-10-23T12:19:23.454+0200    INFO    zapio/writer.go:146     go: downloading github.com/mitchellh/hashstructure v1.1.0
2023-10-23T12:19:24.056+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/contrib/zpages v0.45.0
2023-10-23T12:19:24.056+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/collector/config/configauth v0.87.0
2023-10-23T12:19:24.057+0200    INFO    zapio/writer.go:146     go: downloading contrib.go.opencensus.io/exporter/prometheus v0.4.2
2023-10-23T12:19:24.057+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/otel/exporters/prometheus v0.42.0
2023-10-23T12:19:24.057+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/otel/sdk/metric v1.19.0
2023-10-23T12:19:24.059+0200    INFO    zapio/writer.go:146     go: downloading github.com/davecgh/go-spew v1.1.2-0.20180830191138-d8f796af33cc
2023-10-23T12:19:24.060+0200    INFO    zapio/writer.go:146     go: downloading github.com/pmezard/go-difflib v1.0.1-0.20181226105442-5d4384ee4fb2
2023-10-23T12:19:24.061+0200    INFO    zapio/writer.go:146     go: downloading github.com/spf13/cobra v1.7.0
2023-10-23T12:19:24.061+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/collector/service v0.87.0
2023-10-23T12:19:24.080+0200    INFO    zapio/writer.go:146     go: downloading github.com/cskr/pubsub v1.0.2
2023-10-23T12:19:24.096+0200    INFO    zapio/writer.go:146     go: downloading github.com/knadh/koanf/maps v0.1.1
2023-10-23T12:19:24.108+0200    INFO    zapio/writer.go:146     go: downloading github.com/knadh/koanf/providers/confmap v0.1.0
2023-10-23T12:19:24.112+0200    INFO    zapio/writer.go:146     go: downloading github.com/knadh/koanf/v2 v2.0.1
2023-10-23T12:19:24.116+0200    INFO    zapio/writer.go:146     go: downloading github.com/mitchellh/mapstructure v1.5.1-0.20220423185008-bf980b35cac4
2023-10-23T12:19:24.120+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/collector/config/internal v0.87.0
2023-10-23T12:19:24.122+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.45.0
2023-10-23T12:19:24.123+0200    INFO    zapio/writer.go:146     go: downloading github.com/stretchr/objx v0.5.0
2023-10-23T12:19:24.127+0200    INFO    zapio/writer.go:146     go: downloading github.com/knadh/koanf v1.5.0
2023-10-23T12:19:24.132+0200    INFO    zapio/writer.go:146     go: downloading github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd
2023-10-23T12:19:24.133+0200    INFO    zapio/writer.go:146     go: downloading github.com/modern-go/reflect2 v1.0.2
2023-10-23T12:19:24.146+0200    INFO    zapio/writer.go:146     go: downloading github.com/DataDog/opentelemetry-mapping-go/pkg/quantile v0.8.0
2023-10-23T12:19:24.146+0200    INFO    zapio/writer.go:146     go: downloading github.com/golang/protobuf v1.5.3
2023-10-23T12:19:24.147+0200    INFO    zapio/writer.go:146     go: downloading github.com/patrickmn/go-cache v2.1.0+incompatible
2023-10-23T12:19:24.163+0200    INFO    zapio/writer.go:146     go: downloading github.com/kylelemons/godebug v1.1.0
2023-10-23T12:19:24.176+0200    INFO    zapio/writer.go:146     go: downloading github.com/samber/lo v1.37.0
2023-10-23T12:19:24.199+0200    INFO    zapio/writer.go:146     go: downloading github.com/gofrs/uuid v4.2.0+incompatible
2023-10-23T12:19:24.204+0200    INFO    zapio/writer.go:146     go: downloading github.com/shopspring/decimal v1.3.1
2023-10-23T12:19:24.224+0200    INFO    zapio/writer.go:146     go: downloading github.com/cespare/xxhash/v2 v2.2.0
2023-10-23T12:19:24.224+0200    INFO    zapio/writer.go:146     go: downloading github.com/google/gofuzz v1.2.0
2023-10-23T12:19:24.231+0200    INFO    zapio/writer.go:146     go: downloading cloud.google.com/go/logging v1.7.0
2023-10-23T12:19:24.238+0200    INFO    zapio/writer.go:146     go: downloading cloud.google.com/go/monitoring v1.15.1
2023-10-23T12:19:24.238+0200    INFO    zapio/writer.go:146     go: downloading cloud.google.com/go/trace v1.10.1
2023-10-23T12:19:24.239+0200    INFO    zapio/writer.go:146     go: downloading github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/trace v1.20.0
2023-10-23T12:19:24.240+0200    INFO    zapio/writer.go:146     go: downloading github.com/GoogleCloudPlatform/opentelemetry-operations-go/internal/resourcemapping v0.44.0
2023-10-23T12:19:24.298+0200    INFO    zapio/writer.go:146     go: downloading github.com/googleapis/gax-go/v2 v2.12.0
2023-10-23T12:19:24.301+0200    INFO    zapio/writer.go:146     go: downloading google.golang.org/genproto/googleapis/api v0.0.0-20230913181813-007df8e322eb
2023-10-23T12:19:24.376+0200    INFO    zapio/writer.go:146     go: downloading github.com/go-kit/kit v0.10.0
2023-10-23T12:19:24.386+0200    INFO    zapio/writer.go:146     go: downloading go.uber.org/atomic v1.11.0
2023-10-23T12:19:24.387+0200    INFO    zapio/writer.go:146     go: downloading github.com/pkg/errors v0.9.1
2023-10-23T12:19:24.397+0200    INFO    zapio/writer.go:146     go: downloading github.com/aws/smithy-go v1.15.0
2023-10-23T12:19:24.412+0200    INFO    zapio/writer.go:146     go: downloading github.com/aws/aws-sdk-go-v2/feature/ec2/imds v1.13.12
2023-10-23T12:19:24.464+0200    INFO    zapio/writer.go:146     go: downloading github.com/aws/aws-sdk-go-v2/internal/ini v1.3.44
2023-10-23T12:19:24.464+0200    INFO    zapio/writer.go:146     go: downloading github.com/aws/aws-sdk-go-v2/service/sso v1.15.1
2023-10-23T12:19:24.464+0200    INFO    zapio/writer.go:146     go: downloading github.com/aws/aws-sdk-go-v2/service/ssooidc v1.17.2
2023-10-23T12:19:24.476+0200    INFO    zapio/writer.go:146     go: downloading github.com/aws/aws-sdk-go-v2/aws/protocol/eventstream v1.4.14
2023-10-23T12:19:24.476+0200    INFO    zapio/writer.go:146     go: downloading github.com/aws/aws-sdk-go-v2/internal/configsources v1.1.42
2023-10-23T12:19:24.477+0200    INFO    zapio/writer.go:146     go: downloading github.com/jmespath/go-jmespath v0.4.0
2023-10-23T12:19:24.477+0200    INFO    zapio/writer.go:146     go: downloading github.com/aws/aws-sdk-go-v2/service/internal/presigned-url v1.9.36
2023-10-23T12:19:24.486+0200    INFO    zapio/writer.go:146     go: downloading github.com/mostynb/go-grpc-compression v1.2.1
2023-10-23T12:19:24.489+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc v0.45.0
2023-10-23T12:19:24.493+0200    INFO    zapio/writer.go:146     go: downloading github.com/hailocab/go-hostpool v0.0.0-20160125115350-e80d13ce29ed
2023-10-23T12:19:24.498+0200    INFO    zapio/writer.go:146     go: downloading gopkg.in/inf.v0 v0.9.1
2023-10-23T12:19:24.498+0200    INFO    zapio/writer.go:146     go: downloading github.com/alecthomas/participle/v2 v2.1.0
2023-10-23T12:19:24.509+0200    INFO    zapio/writer.go:146     go: downloading github.com/iancoleman/strcase v0.3.0
2023-10-23T12:19:24.515+0200    INFO    zapio/writer.go:146     go: downloading github.com/elastic/go-structform v0.0.10
2023-10-23T12:19:24.537+0200    INFO    zapio/writer.go:146     go: downloading cloud.google.com/go/iam v1.1.1
2023-10-23T12:19:24.548+0200    INFO    zapio/writer.go:146     go: downloading code.cloudfoundry.org/clock v0.0.0-20180518195852-02e53af36e6c
2023-10-23T12:19:24.562+0200    INFO    zapio/writer.go:146     go: downloading github.com/fatih/color v1.15.0
2023-10-23T12:19:24.562+0200    INFO    zapio/writer.go:146     go: downloading github.com/mattn/go-colorable v0.1.13
2023-10-23T12:19:24.575+0200    INFO    zapio/writer.go:146     go: downloading github.com/mattn/go-isatty v0.0.19
2023-10-23T12:19:24.585+0200    INFO    zapio/writer.go:146     go: downloading github.com/frankban/quicktest v1.14.3
2023-10-23T12:19:24.593+0200    INFO    zapio/writer.go:146     go: downloading github.com/ClickHouse/ch-go v0.58.2
2023-10-23T12:19:24.602+0200    INFO    zapio/writer.go:146     go: downloading github.com/andybalholm/brotli v1.0.5
2023-10-23T12:19:24.613+0200    INFO    zapio/writer.go:146     go: downloading github.com/uber/jaeger-client-go v2.30.0+incompatible
2023-10-23T12:19:24.622+0200    INFO    zapio/writer.go:146     go: downloading github.com/eapache/go-resiliency v1.4.0
2023-10-23T12:19:24.622+0200    INFO    zapio/writer.go:146     go: downloading github.com/xdg-go/scram v1.1.2
2023-10-23T12:19:24.662+0200    INFO    zapio/writer.go:146     go: downloading github.com/eapache/go-xerial-snappy v0.0.0-20230731223053-c322873962e3
2023-10-23T12:19:24.668+0200    INFO    zapio/writer.go:146     go: downloading github.com/eapache/queue v1.1.0
2023-10-23T12:19:24.685+0200    INFO    zapio/writer.go:146     go: downloading github.com/hashicorp/go-multierror v1.1.1
2023-10-23T12:19:24.694+0200    INFO    zapio/writer.go:146     go: downloading github.com/jcmturner/gofork v1.7.6
2023-10-23T12:19:24.704+0200    INFO    zapio/writer.go:146     go: downloading github.com/jcmturner/gokrb5/v8 v8.4.4
2023-10-23T12:19:24.783+0200    INFO    zapio/writer.go:146     go: downloading github.com/pierrec/lz4/v4 v4.1.18
2023-10-23T12:19:24.783+0200    INFO    zapio/writer.go:146     go: downloading github.com/rcrowley/go-metrics v0.0.0-20201227073835-cf1acfcdf475
2023-10-23T12:19:24.783+0200    INFO    zapio/writer.go:146     go: downloading github.com/fortytw2/leaktest v1.3.0
2023-10-23T12:19:24.783+0200    INFO    zapio/writer.go:146     go: downloading golang.org/x/sync v0.4.0
2023-10-23T12:19:24.783+0200    INFO    zapio/writer.go:146     go: downloading github.com/go-logfmt/logfmt v0.6.0
2023-10-23T12:19:24.794+0200    INFO    zapio/writer.go:146     go: downloading github.com/linkedin/goavro/v2 v2.9.8
2023-10-23T12:19:24.802+0200    INFO    zapio/writer.go:146     go: downloading github.com/sirupsen/logrus v1.9.0
2023-10-23T12:19:24.805+0200    INFO    zapio/writer.go:146     go: downloading github.com/beorn7/perks v1.0.1
2023-10-23T12:19:24.810+0200    INFO    zapio/writer.go:146     go: downloading github.com/prometheus/procfs v0.11.1
2023-10-23T12:19:24.811+0200    INFO    zapio/writer.go:146     go: downloading github.com/GehirnInc/crypt v0.0.0-20200316065508-bb7000b8a962
2023-10-23T12:19:24.824+0200    INFO    zapio/writer.go:146     go: downloading github.com/pquerna/cachecontrol v0.1.0
2023-10-23T12:19:24.833+0200    INFO    zapio/writer.go:146     go: downloading github.com/vincent-petithory/dataurl v1.0.0
2023-10-23T12:19:24.865+0200    INFO    zapio/writer.go:146     go: downloading sigs.k8s.io/yaml v1.3.0
2023-10-23T12:19:24.865+0200    INFO    zapio/writer.go:146     go: downloading k8s.io/klog/v2 v2.100.1
2023-10-23T12:19:24.866+0200    INFO    zapio/writer.go:146     go: downloading sigs.k8s.io/structured-merge-diff/v4 v4.3.0
2023-10-23T12:19:24.872+0200    INFO    zapio/writer.go:146     go: downloading github.com/onsi/ginkgo v1.16.5
2023-10-23T12:19:24.880+0200    INFO    zapio/writer.go:146     go: downloading github.com/onsi/gomega v1.17.0
2023-10-23T12:19:24.890+0200    INFO    zapio/writer.go:146     go: downloading github.com/evanphx/json-patch/v5 v5.6.0
2023-10-23T12:19:24.906+0200    INFO    zapio/writer.go:146     go: downloading github.com/evanphx/json-patch v5.6.0+incompatible
2023-10-23T12:19:24.908+0200    INFO    zapio/writer.go:146     go: downloading k8s.io/klog v1.0.0
2023-10-23T12:19:24.918+0200    INFO    zapio/writer.go:146     go: downloading github.com/tidwall/gjson v1.10.2
2023-10-23T12:19:24.919+0200    INFO    zapio/writer.go:146     go: downloading github.com/tidwall/tinylru v1.1.0
2023-10-23T12:19:24.922+0200    INFO    zapio/writer.go:146     go: downloading github.com/DataDog/datadog-agent/pkg/obfuscate v0.48.0-beta.1
2023-10-23T12:19:24.933+0200    INFO    zapio/writer.go:146     go: downloading github.com/DataDog/datadog-agent/pkg/remoteconfig/state v0.48.0-beta.1
2023-10-23T12:19:24.937+0200    INFO    zapio/writer.go:146     go: downloading github.com/DataDog/datadog-agent/pkg/util/cgroups v0.48.0-beta.1
2023-10-23T12:19:24.941+0200    INFO    zapio/writer.go:146     go: downloading github.com/DataDog/datadog-agent/pkg/util/log v0.48.0-beta.1
2023-10-23T12:19:24.967+0200    INFO    zapio/writer.go:146     go: downloading github.com/Microsoft/go-winio v0.6.1
2023-10-23T12:19:24.971+0200    INFO    zapio/writer.go:146     go: downloading github.com/zorkian/go-datadog-api v2.30.0+incompatible
2023-10-23T12:19:24.985+0200    INFO    zapio/writer.go:146     go: downloading github.com/caio/go-tdigest/v4 v4.0.1
2023-10-23T12:19:24.986+0200    INFO    zapio/writer.go:146     go: downloading github.com/alecthomas/units v0.0.0-20211218093645-b94a6e3cc137
2023-10-23T12:19:24.988+0200    INFO    zapio/writer.go:146     go: downloading github.com/grafana/regexp v0.0.0-20221122212121-6b5c0a4cb7fd
2023-10-23T12:19:24.993+0200    INFO    zapio/writer.go:146     go: downloading github.com/prometheus/common/sigv4 v0.1.0
2023-10-23T12:19:25.001+0200    INFO    zapio/writer.go:146     go: downloading github.com/gobwas/glob v0.2.3
2023-10-23T12:19:25.002+0200    INFO    zapio/writer.go:146     go: downloading gopkg.in/square/go-jose.v2 v2.5.1
2023-10-23T12:19:25.023+0200    INFO    zapio/writer.go:146     go: downloading github.com/power-devops/perfstat v0.0.0-20220216144756-c35f1ee13d7c
2023-10-23T12:19:25.040+0200    INFO    zapio/writer.go:146     go: downloading github.com/tklauser/go-sysconf v0.3.12
2023-10-23T12:19:25.064+0200    INFO    zapio/writer.go:146     go: downloading github.com/imdario/mergo v0.3.16
2023-10-23T12:19:25.064+0200    INFO    zapio/writer.go:146     go: downloading github.com/spf13/pflag v1.0.5
2023-10-23T12:19:25.069+0200    INFO    zapio/writer.go:146     go: downloading golang.org/x/term v0.13.0
2023-10-23T12:19:25.081+0200    INFO    zapio/writer.go:146     go: downloading github.com/spf13/viper v1.16.0
2023-10-23T12:19:25.091+0200    INFO    zapio/writer.go:146     go: downloading github.com/gogo/googleapis v1.4.1
2023-10-23T12:19:25.102+0200    INFO    zapio/writer.go:146     go: downloading github.com/golang/groupcache v0.0.0-20210331224755-41bb18bfe9da
2023-10-23T12:19:25.105+0200    INFO    zapio/writer.go:146     go: downloading github.com/jackc/pgconn v1.14.0
2023-10-23T12:19:25.106+0200    INFO    zapio/writer.go:146     go: downloading github.com/jackc/pgtype v1.14.0
2023-10-23T12:19:25.108+0200    INFO    zapio/writer.go:146     go: downloading dario.cat/mergo v1.0.0
2023-10-23T12:19:25.129+0200    INFO    zapio/writer.go:146     go: downloading github.com/containerd/containerd v1.7.6
2023-10-23T12:19:25.130+0200    INFO    zapio/writer.go:146     go: downloading github.com/cpuguy83/dockercfg v0.3.1
2023-10-23T12:19:25.134+0200    INFO    zapio/writer.go:146     go: downloading github.com/moby/term v0.5.0
2023-10-23T12:19:25.136+0200    INFO    zapio/writer.go:146     go: downloading github.com/opencontainers/image-spec v1.1.0-rc4
2023-10-23T12:19:25.144+0200    INFO    zapio/writer.go:146     go: downloading github.com/Azure/go-autorest/autorest v0.11.29
2023-10-23T12:19:25.151+0200    INFO    zapio/writer.go:146     go: downloading github.com/Azure/go-autorest/autorest/to v0.4.0
2023-10-23T12:19:25.152+0200    INFO    zapio/writer.go:146     go: downloading github.com/devigned/tab v0.1.1
2023-10-23T12:19:25.152+0200    INFO    zapio/writer.go:146     go: downloading github.com/Azure/go-autorest/autorest/date v0.3.0
2023-10-23T12:19:25.158+0200    INFO    zapio/writer.go:146     go: downloading github.com/jpillora/backoff v1.0.0
2023-10-23T12:19:25.177+0200    INFO    zapio/writer.go:146     go: downloading github.com/yuin/gopher-lua v0.0.0-20220504180219-658193537a64
2023-10-23T12:19:25.181+0200    INFO    zapio/writer.go:146     go: downloading github.com/goccy/go-json v0.10.2
2023-10-23T12:19:25.182+0200    INFO    zapio/writer.go:146     go: downloading github.com/docker/go-units v0.5.0
2023-10-23T12:19:25.195+0200    INFO    zapio/writer.go:146     go: downloading github.com/Azure/go-autorest v14.2.0+incompatible
2023-10-23T12:19:25.199+0200    INFO    zapio/writer.go:146     go: downloading code.cloudfoundry.org/go-diodes v0.0.0-20211115184647-b584dd5df32c
2023-10-23T12:19:25.235+0200    INFO    zapio/writer.go:146     go: downloading code.cloudfoundry.org/rfc5424 v0.0.0-20201103192249-000122071b78
2023-10-23T12:19:25.249+0200    INFO    zapio/writer.go:146     go: downloading github.com/facebook/time v0.0.0-20220713225404-f7a0d7702d50
2023-10-23T12:19:25.257+0200    INFO    zapio/writer.go:146     go: downloading github.com/go-logr/logr v1.2.4
2023-10-23T12:19:25.258+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/internal/metadataproviders v0.87.0
2023-10-23T12:19:25.271+0200    INFO    zapio/writer.go:146     go: downloading github.com/hashicorp/consul/api v1.25.1
2023-10-23T12:19:25.327+0200    INFO    zapio/writer.go:146     go: downloading cloud.google.com/go/compute/metadata v0.2.4-0.20230617002413-005d2dfb6b68
2023-10-23T12:19:25.337+0200    INFO    zapio/writer.go:146     go: downloading github.com/GoogleCloudPlatform/opentelemetry-operations-go/detectors/gcp v1.20.0
2023-10-23T12:19:25.338+0200    INFO    zapio/writer.go:146     go: downloading google.golang.org/appengine v1.6.7
2023-10-23T12:19:25.338+0200    INFO    zapio/writer.go:146     go: downloading github.com/vmihailenco/tagparser v0.1.2
2023-10-23T12:19:25.338+0200    INFO    zapio/writer.go:146     go: downloading github.com/philhofer/fwd v1.1.2
2023-10-23T12:19:25.354+0200    INFO    zapio/writer.go:146     go: downloading github.com/google/gnostic-models v0.6.8
2023-10-23T12:19:25.379+0200    INFO    zapio/writer.go:146     go: downloading k8s.io/kube-openapi v0.0.0-20230717233707-2695361300d9
2023-10-23T12:19:25.384+0200    INFO    zapio/writer.go:146     go: downloading cloud.google.com/go/spanner v1.50.0
2023-10-23T12:19:25.457+0200    INFO    zapio/writer.go:146     go: downloading github.com/buger/jsonparser v1.1.1
2023-10-23T12:19:25.501+0200    INFO    zapio/writer.go:146     go: downloading golang.org/x/text v0.13.0
2023-10-23T12:19:25.533+0200    INFO    zapio/writer.go:146     go: downloading cloud.google.com/go/compute v1.23.0
2023-10-23T12:19:25.945+0200    INFO    zapio/writer.go:146     go: downloading github.com/google/cadvisor v0.47.3
2023-10-23T12:19:26.725+0200    INFO    zapio/writer.go:146     go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/internal/aws/k8s v0.87.0
2023-10-23T12:19:26.804+0200    INFO    zapio/writer.go:146     go: downloading k8s.io/kubelet v0.28.2
2023-10-23T12:19:26.804+0200    INFO    zapio/writer.go:146     go: downloading github.com/vmware/go-vmware-nsxt v0.0.0-20220328155605-f49a14c1ef5f
2023-10-23T12:19:26.902+0200    INFO    zapio/writer.go:146     go: downloading github.com/mongodb-forks/digest v1.0.5
2023-10-23T12:19:26.946+0200    INFO    zapio/writer.go:146     go: downloading github.com/google/go-querystring v1.1.0
2023-10-23T12:19:26.960+0200    INFO    zapio/writer.go:146     go: downloading github.com/youmark/pkcs8 v0.0.0-20181117223130-1be2e3e5546d
2023-10-23T12:19:26.976+0200    INFO    zapio/writer.go:146     go: downloading github.com/mwitkow/go-conntrack v0.0.0-20190716064945-2f068394615f
2023-10-23T12:19:26.996+0200    INFO    zapio/writer.go:146     go: downloading gonum.org/v1/gonum v0.14.0
2023-10-23T12:19:27.112+0200    INFO    zapio/writer.go:146     go: downloading github.com/kr/fs v0.1.0
2023-10-23T12:19:27.127+0200    INFO    zapio/writer.go:146     go: downloading github.com/golang-sql/civil v0.0.0-20190719163853-cb61b32ac6fe
2023-10-23T12:19:27.172+0200    INFO    zapio/writer.go:146     go: downloading github.com/golang-sql/sqlexp v0.1.0
2023-10-23T12:19:27.186+0200    INFO    zapio/writer.go:146     go: downloading github.com/99designs/keyring v1.2.2
2023-10-23T12:19:27.254+0200    INFO    zapio/writer.go:146     go: downloading github.com/apache/arrow/go/v12 v12.0.1
2023-10-23T12:19:27.998+0200    INFO    zapio/writer.go:146     go: downloading github.com/aws/aws-sdk-go-v2/feature/s3/manager v1.11.59
2023-10-23T12:19:27.998+0200    INFO    zapio/writer.go:146     go: downloading github.com/aws/aws-sdk-go-v2/service/s3 v1.31.0
2023-10-23T12:19:27.998+0200    INFO    zapio/writer.go:146     go: downloading github.com/form3tech-oss/jwt-go v3.2.5+incompatible
2023-10-23T12:19:27.998+0200    INFO    zapio/writer.go:146     go: downloading github.com/gabriel-vasile/mimetype v1.4.2
2023-10-23T12:19:27.999+0200    INFO    zapio/writer.go:146     go: downloading github.com/pkg/browser v0.0.0-20210911075715-681adbf594b8
2023-10-23T12:19:27.999+0200    INFO    zapio/writer.go:146     go: downloading github.com/basgys/goxml2json v1.1.0
2023-10-23T12:19:28.016+0200    INFO    zapio/writer.go:146     go: downloading github.com/matttproud/golang_protobuf_extensions v1.0.4
2023-10-23T12:19:28.027+0200    INFO    zapio/writer.go:146     go: downloading github.com/prometheus/statsd_exporter v0.22.7
2023-10-23T12:19:28.032+0200    INFO    zapio/writer.go:146     go: downloading github.com/mitchellh/copystructure v1.2.0
2023-10-23T12:19:28.035+0200    INFO    zapio/writer.go:146     go: downloading github.com/inconshreveable/mousetrap v1.1.0
2023-10-23T12:19:28.042+0200    INFO    zapio/writer.go:146     go: downloading github.com/go-logr/stdr v1.2.2
2023-10-23T12:19:28.046+0200    INFO    zapio/writer.go:146     go: downloading github.com/felixge/httpsnoop v1.0.3
2023-10-23T12:19:28.053+0200    INFO    zapio/writer.go:146     go: downloading github.com/Azure/azure-pipeline-go v0.2.3
2023-10-23T12:19:28.057+0200    INFO    zapio/writer.go:146     go: downloading github.com/Azure/azure-storage-queue-go v0.0.0-20191125232315-636801874cdd
2023-10-23T12:19:28.063+0200    INFO    zapio/writer.go:146     go: downloading github.com/dustin/go-humanize v1.0.1
2023-10-23T12:19:28.071+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/contrib/propagators/b3 v1.19.0
2023-10-23T12:19:28.079+0200    INFO    zapio/writer.go:146     go: downloading cloud.google.com/go/longrunning v0.5.1
2023-10-23T12:19:28.098+0200    INFO    zapio/writer.go:146     go: downloading github.com/aws/aws-sdk-go-v2/internal/endpoints/v2 v2.4.36
2023-10-23T12:19:28.103+0200    INFO    zapio/writer.go:146     go: downloading github.com/paulmach/orb v0.10.0
2023-10-23T12:19:28.111+0200    INFO    zapio/writer.go:146     go: downloading github.com/xdg-go/pbkdf2 v1.0.0
2023-10-23T12:19:28.121+0200    INFO    zapio/writer.go:146     go: downloading github.com/xdg-go/stringprep v1.0.4
2023-10-23T12:19:28.124+0200    INFO    zapio/writer.go:146     go: downloading github.com/hashicorp/errwrap v1.1.0
2023-10-23T12:19:28.137+0200    INFO    zapio/writer.go:146     go: downloading github.com/opentracing/opentracing-go v1.2.0
2023-10-23T12:19:28.160+0200    INFO    zapio/writer.go:146     go: downloading github.com/uber/jaeger-lib v2.4.1+incompatible
2023-10-23T12:19:28.161+0200    INFO    zapio/writer.go:146     go: downloading github.com/jcmturner/dnsutils/v2 v2.0.0
2023-10-23T12:19:28.176+0200    INFO    zapio/writer.go:146     go: downloading github.com/hashicorp/go-uuid v1.0.3
2023-10-23T12:19:28.191+0200    INFO    zapio/writer.go:146     go: downloading github.com/spaolacci/murmur3 v1.1.0
2023-10-23T12:19:28.198+0200    INFO    zapio/writer.go:146     go: downloading github.com/AthenZ/athenz v1.10.39
2023-10-23T12:19:28.202+0200    INFO    zapio/writer.go:146     go: downloading github.com/apache/pulsar-client-go/oauth2 v0.0.0-20220120090717-25e59572242e
2023-10-23T12:19:28.272+0200    INFO    zapio/writer.go:146     go: downloading github.com/DataDog/zstd v1.5.2
2023-10-23T12:19:28.408+0200    INFO    zapio/writer.go:146     go: downloading github.com/go-faster/city v1.0.1
2023-10-23T12:19:28.435+0200    INFO    zapio/writer.go:146     go: downloading github.com/go-faster/errors v0.6.1
2023-10-23T12:19:28.459+0200    INFO    zapio/writer.go:146     go: downloading github.com/segmentio/asm v1.2.0
2023-10-23T12:19:28.475+0200    INFO    zapio/writer.go:146     go: downloading sigs.k8s.io/json v0.0.0-20221116044647-bc3834ca7abd
2023-10-23T12:19:28.516+0200    INFO    zapio/writer.go:146     go: downloading github.com/DataDog/datadog-agent/pkg/util/scrubber v0.48.0-beta.1
2023-10-23T12:19:28.517+0200    INFO    zapio/writer.go:146     go: downloading github.com/DataDog/datadog-go/v5 v5.1.1
2023-10-23T12:19:28.708+0200    INFO    zapio/writer.go:146     go: downloading github.com/cihub/seelog v0.0.0-20170130134532-f561c5e57575
2023-10-23T12:19:28.717+0200    INFO    zapio/writer.go:146     go: downloading github.com/golang/mock v1.6.0
2023-10-23T12:19:28.772+0200    INFO    zapio/writer.go:146     go: downloading github.com/DataDog/datadog-agent/pkg/util/pointer v0.48.0-beta.1
2023-10-23T12:19:28.787+0200    INFO    zapio/writer.go:146     go: downloading github.com/DataDog/gohai v0.0.0-20220718130825-1776f9beb9cc
2023-10-23T12:19:28.897+0200    INFO    zapio/writer.go:146     go: downloading github.com/tidwall/match v1.1.1
2023-10-23T12:19:28.897+0200    INFO    zapio/writer.go:146     go: downloading github.com/tidwall/pretty v1.2.0
2023-10-23T12:19:28.925+0200    INFO    zapio/writer.go:146     go: downloading github.com/DataDog/go-tuf v1.0.1-0.5.2
2023-10-23T12:19:29.357+0200    INFO    zapio/writer.go:146     go: downloading github.com/secure-systems-lab/go-securesystemslib v0.7.0
2023-10-23T12:19:29.377+0200    INFO    zapio/writer.go:146     go: downloading github.com/Azure/go-autorest/autorest/adal v0.9.23
2023-10-23T12:19:29.395+0200    INFO    zapio/writer.go:146     go: downloading github.com/digitalocean/godo v1.99.0
2023-10-23T12:19:29.490+0200    INFO    zapio/writer.go:146     go: downloading github.com/miekg/dns v1.1.55
2023-10-23T12:19:29.545+0200    INFO    zapio/writer.go:146     go: downloading github.com/hetznercloud/hcloud-go/v2 v2.0.0
2023-10-23T12:19:29.623+0200    INFO    zapio/writer.go:146     go: downloading github.com/ionos-cloud/sdk-go/v6 v6.1.8
2023-10-23T12:19:29.915+0200    INFO    zapio/writer.go:146     go: downloading github.com/linode/linodego v1.19.0
2023-10-23T12:19:29.989+0200    INFO    zapio/writer.go:146     go: downloading github.com/hashicorp/nomad/api v0.0.0-20230718173136-3a687930bd3e
2023-10-23T12:19:30.068+0200    INFO    zapio/writer.go:146     go: downloading github.com/gophercloud/gophercloud v1.5.0
2023-10-23T12:19:30.848+0200    INFO    zapio/writer.go:146     go: downloading github.com/ovh/go-ovh v1.4.1
2023-10-23T12:19:30.848+0200    INFO    zapio/writer.go:146     go: downloading github.com/scaleway/scaleway-sdk-go v1.0.0-beta.20
2023-10-23T12:19:31.177+0200    INFO    zapio/writer.go:146     go: downloading github.com/kolo/xmlrpc v0.0.0-20220921171641-a4b6fa1dd06b
2023-10-23T12:19:31.178+0200    INFO    zapio/writer.go:146     go: downloading github.com/vultr/govultr/v2 v2.17.2
2023-10-23T12:19:31.215+0200    INFO    zapio/writer.go:146     go: downloading github.com/envoyproxy/go-control-plane v0.11.1
2023-10-23T12:19:32.181+0200    INFO    zapio/writer.go:146     go: downloading github.com/envoyproxy/protoc-gen-validate v1.0.2
2023-10-23T12:19:32.181+0200    INFO    zapio/writer.go:146     go: downloading github.com/go-zookeeper/zk v1.0.3
2023-10-23T12:19:32.205+0200    INFO    zapio/writer.go:146     go: downloading gotest.tools/v3 v3.0.3
2023-10-23T12:19:32.298+0200    INFO    zapio/writer.go:146     go: downloading github.com/docker/distribution v2.8.2+incompatible
2023-10-23T12:19:32.298+0200    INFO    zapio/writer.go:146     go: downloading github.com/opencontainers/go-digest v1.0.0
2023-10-23T12:19:32.411+0200    INFO    zapio/writer.go:146     go: downloading github.com/yusufpapurcu/wmi v1.2.3
2023-10-23T12:19:32.411+0200    INFO    zapio/writer.go:146     go: downloading github.com/lufia/plan9stats v0.0.0-20220913051719-115f729f3c8c
2023-10-23T12:19:32.411+0200    INFO    zapio/writer.go:146     go: downloading github.com/shoenig/go-m1cpu v0.1.6
2023-10-23T12:19:32.422+0200    INFO    zapio/writer.go:146     go: downloading github.com/containerd/cgroups v1.0.4
2023-10-23T12:19:32.422+0200    INFO    zapio/writer.go:146     go: downloading github.com/karrick/godirwalk v1.17.0
2023-10-23T12:19:32.449+0200    INFO    zapio/writer.go:146     go: downloading golang.org/x/tools v0.14.0
2023-10-23T12:19:32.458+0200    INFO    zapio/writer.go:146     go: downloading github.com/outcaste-io/ristretto v0.2.1
2023-10-23T12:19:32.494+0200    INFO    zapio/writer.go:146     go: downloading github.com/tklauser/numcpus v0.6.1
2023-10-23T12:19:32.506+0200    INFO    zapio/writer.go:146     go: downloading github.com/jackc/pgio v1.0.0
2023-10-23T12:19:32.517+0200    INFO    zapio/writer.go:146     go: downloading github.com/jackc/pgproto3/v2 v2.3.2
2023-10-23T12:19:32.521+0200    INFO    zapio/writer.go:146     go: downloading github.com/moby/patternmatcher v0.5.0
2023-10-23T12:19:32.539+0200    INFO    zapio/writer.go:146     go: downloading github.com/moby/sys/sequential v0.5.0
2023-10-23T12:19:32.549+0200    INFO    zapio/writer.go:146     go: downloading github.com/morikuni/aec v1.0.0
2023-10-23T12:19:32.555+0200    INFO    zapio/writer.go:146     go: downloading github.com/magiconair/properties v1.8.7
2023-10-23T12:19:32.572+0200    INFO    zapio/writer.go:146     go: downloading github.com/spf13/afero v1.9.5
2023-10-23T12:19:32.578+0200    INFO    zapio/writer.go:146     go: downloading github.com/spf13/jwalterweatherman v1.1.0
2023-10-23T12:19:32.590+0200    INFO    zapio/writer.go:146     go: downloading github.com/jackc/chunkreader/v2 v2.0.1
2023-10-23T12:19:32.602+0200    INFO    zapio/writer.go:146     go: downloading github.com/jackc/pgpassfile v1.0.0
2023-10-23T12:19:32.611+0200    INFO    zapio/writer.go:146     go: downloading github.com/jackc/pgservicefile v0.0.0-20221227161230-091c0ba34f0a
2023-10-23T12:19:32.617+0200    INFO    zapio/writer.go:146     go: downloading github.com/influxdata/go-syslog/v3 v3.0.1-0.20210608084020-ac565dc76ba6
2023-10-23T12:19:32.720+0200    INFO    zapio/writer.go:146     go: downloading github.com/Azure/go-autorest/tracing v0.6.0
2023-10-23T12:19:32.720+0200    INFO    zapio/writer.go:146     go: downloading github.com/Azure/go-autorest/logger v0.2.1
2023-10-23T12:19:32.731+0200    INFO    zapio/writer.go:146     go: downloading github.com/Showmax/go-fqdn v1.0.0
2023-10-23T12:19:32.731+0200    INFO    zapio/writer.go:146     go: downloading github.com/deepmap/oapi-codegen v1.8.2
2023-10-23T12:19:32.782+0200    INFO    zapio/writer.go:146     go: downloading github.com/hashicorp/go-cleanhttp v0.5.2
2023-10-23T12:19:32.800+0200    INFO    zapio/writer.go:146     go: downloading github.com/hashicorp/go-rootcerts v1.0.2
2023-10-23T12:19:32.819+0200    INFO    zapio/writer.go:146     go: downloading github.com/hashicorp/serf v0.10.1
2023-10-23T12:19:32.966+0200    INFO    zapio/writer.go:146     go: downloading github.com/ReneKroon/ttlcache/v2 v2.11.0
2023-10-23T12:19:32.967+0200    INFO    zapio/writer.go:146     go: downloading golang.org/x/xerrors v0.0.0-20220907171357-04be3eba64a2
2023-10-23T12:19:32.985+0200    INFO    zapio/writer.go:146     go: downloading github.com/Microsoft/hcsshim v0.11.0
2023-10-23T12:19:33.031+0200    INFO    zapio/writer.go:146     go: downloading github.com/go-openapi/swag v0.22.4
2023-10-23T12:19:33.071+0200    INFO    zapio/writer.go:146     go: downloading github.com/opencontainers/runc v1.1.5
2023-10-23T12:19:33.271+0200    INFO    zapio/writer.go:146     go: downloading github.com/opencontainers/runtime-spec v1.1.0-rc.3
2023-10-23T12:19:33.274+0200    INFO    zapio/writer.go:146     go: downloading github.com/blang/semver v3.5.1+incompatible
2023-10-23T12:19:33.291+0200    INFO    zapio/writer.go:146     go: downloading github.com/google/s2a-go v0.1.7
2023-10-23T12:19:33.382+0200    INFO    zapio/writer.go:146     go: downloading github.com/montanaflynn/stats v0.0.0-20171201202039-1bf9dbcd8cbe
2023-10-23T12:19:33.388+0200    INFO    zapio/writer.go:146     go: downloading github.com/leoluk/perflib_exporter v0.2.1
2023-10-23T12:19:33.389+0200    INFO    zapio/writer.go:146     go: downloading github.com/99designs/go-keychain v0.0.0-20191008050251-8e49817e8af4
2023-10-23T12:19:33.436+0200    INFO    zapio/writer.go:146     go: downloading github.com/danieljoos/wincred v1.1.2
2023-10-23T12:19:33.450+0200    INFO    zapio/writer.go:146     go: downloading github.com/dvsekhvalnov/jose2go v1.5.0
2023-10-23T12:19:33.501+0200    INFO    zapio/writer.go:146     go: downloading github.com/godbus/dbus v0.0.0-20190726142602-4481cbc300e2
2023-10-23T12:19:33.542+0200    INFO    zapio/writer.go:146     go: downloading github.com/gsterjov/go-libsecret v0.0.0-20161001094733-a6f4afe4910c
2023-10-23T12:19:33.553+0200    INFO    zapio/writer.go:146     go: downloading github.com/mtibben/percent v0.2.1
2023-10-23T12:19:33.566+0200    INFO    zapio/writer.go:146     go: downloading github.com/klauspost/cpuid/v2 v2.2.3
2023-10-23T12:19:33.592+0200    INFO    zapio/writer.go:146     go: downloading github.com/google/flatbuffers v23.1.21+incompatible
2023-10-23T12:19:33.692+0200    INFO    zapio/writer.go:146     go: downloading github.com/mitchellh/reflectwalk v1.0.2
2023-10-23T12:19:33.703+0200    INFO    zapio/writer.go:146     go: downloading github.com/mattn/go-ieproxy v0.0.1
2023-10-23T12:19:33.731+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/otel/bridge/opencensus v0.42.0
2023-10-23T12:19:33.794+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetricgrpc v0.42.0
2023-10-23T12:19:33.820+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/otel/exporters/otlp/otlpmetric v0.42.0
2023-10-23T12:19:33.833+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetrichttp v0.42.0
2023-10-23T12:19:33.884+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc v1.19.0
2023-10-23T12:19:33.896+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp v1.19.0
2023-10-23T12:19:33.906+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/otel/exporters/otlp/otlptrace v1.19.0
2023-10-23T12:19:33.921+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/otel/exporters/stdout/stdoutmetric v0.42.0
2023-10-23T12:19:33.921+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/otel/exporters/stdout/stdouttrace v1.19.0
2023-10-23T12:19:33.933+0200    INFO    zapio/writer.go:146     go: downloading github.com/cncf/xds/go v0.0.0-20230607035331-e9ce68804cb4
2023-10-23T12:19:33.934+0200    INFO    zapio/writer.go:146     go: downloading github.com/googleapis/enterprise-certificate-proxy v0.3.1
2023-10-23T12:19:33.982+0200    INFO    zapio/writer.go:146     go: downloading github.com/aws/aws-sdk-go-v2/internal/v4a v1.0.23
2023-10-23T12:19:33.992+0200    INFO    zapio/writer.go:146     go: downloading github.com/aws/aws-sdk-go-v2/service/internal/accept-encoding v1.9.11
2023-10-23T12:19:34.007+0200    INFO    zapio/writer.go:146     go: downloading github.com/aws/aws-sdk-go-v2/service/internal/checksum v1.1.26
2023-10-23T12:19:34.032+0200    INFO    zapio/writer.go:146     go: downloading github.com/aws/aws-sdk-go-v2/service/internal/s3shared v1.14.0
2023-10-23T12:19:34.081+0200    INFO    zapio/writer.go:146     go: downloading github.com/jcmturner/rpc/v2 v2.0.3
2023-10-23T12:19:34.108+0200    INFO    zapio/writer.go:146     go: downloading github.com/golang-jwt/jwt v3.2.2+incompatible
2023-10-23T12:19:34.110+0200    INFO    zapio/writer.go:146     go: downloading github.com/golang-jwt/jwt/v4 v4.5.0
2023-10-23T12:19:34.137+0200    INFO    zapio/writer.go:146     go: downloading github.com/go-resty/resty/v2 v2.7.0
2023-10-23T12:19:34.141+0200    INFO    zapio/writer.go:146     go: downloading gopkg.in/ini.v1 v1.67.0
2023-10-23T12:19:34.172+0200    INFO    zapio/writer.go:146     go: downloading github.com/gorilla/websocket v1.5.0
2023-10-23T12:19:34.215+0200    INFO    zapio/writer.go:146     go: downloading github.com/hashicorp/cronexpr v1.1.2
2023-10-23T12:19:34.234+0200    INFO    zapio/writer.go:146     go: downloading github.com/hashicorp/go-retryablehttp v0.7.4
2023-10-23T12:19:34.249+0200    INFO    zapio/writer.go:146     go: downloading github.com/ardielle/ardielle-go v1.5.2
2023-10-23T12:19:34.249+0200    INFO    zapio/writer.go:146     go: downloading github.com/go-ole/go-ole v1.2.6
2023-10-23T12:19:34.298+0200    INFO    zapio/writer.go:146     go: downloading github.com/coreos/go-systemd/v22 v22.5.0
2023-10-23T12:19:34.342+0200    INFO    zapio/writer.go:146     go: downloading github.com/godbus/dbus/v5 v5.0.6
2023-10-23T12:19:34.405+0200    INFO    zapio/writer.go:146     go: downloading github.com/munnerz/goautoneg v0.0.0-20191010083416-a7dc8b61c822
2023-10-23T12:19:34.415+0200    INFO    zapio/writer.go:146     go: downloading github.com/subosito/gotenv v1.4.2
2023-10-23T12:19:34.432+0200    INFO    zapio/writer.go:146     go: downloading github.com/hashicorp/hcl v1.0.0
2023-10-23T12:19:34.476+0200    INFO    zapio/writer.go:146     go: downloading github.com/pelletier/go-toml/v2 v2.0.8
2023-10-23T12:19:34.477+0200    INFO    zapio/writer.go:146     go: downloading github.com/Azure/go-ansiterm v0.0.0-20230124172434-306776ec8161
2023-10-23T12:19:34.496+0200    INFO    zapio/writer.go:146     go: downloading github.com/leodido/ragel-machinery v0.0.0-20181214104525-299bdde78165
2023-10-23T12:19:34.517+0200    INFO    zapio/writer.go:146     go: downloading github.com/nxadm/tail v1.4.8
2023-10-23T12:19:34.537+0200    INFO    zapio/writer.go:146     go: downloading github.com/go-openapi/jsonreference v0.20.2
2023-10-23T12:19:34.549+0200    INFO    zapio/writer.go:146     go: downloading github.com/mitchellh/go-homedir v1.1.0
2023-10-23T12:19:34.549+0200    INFO    zapio/writer.go:146     go: downloading github.com/armon/go-metrics v0.4.1
2023-10-23T12:19:34.560+0200    INFO    zapio/writer.go:146     go: downloading github.com/mailru/easyjson v0.7.7
2023-10-23T12:19:34.577+0200    INFO    zapio/writer.go:146     go: downloading github.com/mistifyio/go-zfs v2.1.2-0.20190413222219-f784269be439+incompatible
2023-10-23T12:19:34.600+0200    INFO    zapio/writer.go:146     go: downloading github.com/moby/sys/mountinfo v0.6.2
2023-10-23T12:19:34.600+0200    INFO    zapio/writer.go:146     go: downloading github.com/containerd/ttrpc v1.1.0
2023-10-23T12:19:34.624+0200    INFO    zapio/writer.go:146     go: downloading github.com/cyphar/filepath-securejoin v0.2.4
2023-10-23T12:19:34.636+0200    INFO    zapio/writer.go:146     go: downloading github.com/euank/go-kmsg-parser v2.0.0+incompatible
2023-10-23T12:19:34.660+0200    INFO    zapio/writer.go:146     go: downloading github.com/dennwc/varint v1.0.0
2023-10-23T12:19:34.674+0200    INFO    zapio/writer.go:146     go: downloading github.com/zeebo/xxh3 v1.0.2
2023-10-23T12:19:34.674+0200    INFO    zapio/writer.go:146     go: downloading github.com/JohnCGriffin/overflow v0.0.0-20211019200055-46fa312c352c
2023-10-23T12:19:34.687+0200    INFO    zapio/writer.go:146     go: downloading go.opentelemetry.io/proto/otlp v1.0.0
2023-10-23T12:19:34.724+0200    INFO    zapio/writer.go:146     go: downloading github.com/jcmturner/aescts/v2 v2.0.0
2023-10-23T12:19:34.738+0200    INFO    zapio/writer.go:146     go: downloading github.com/emicklei/go-restful/v3 v3.10.2
2023-10-23T12:19:34.738+0200    INFO    zapio/writer.go:146     go: downloading github.com/go-openapi/jsonpointer v0.20.0
2023-10-23T12:19:34.740+0200    INFO    zapio/writer.go:146     go: downloading github.com/hashicorp/go-immutable-radix v1.3.1
2023-10-23T12:19:34.754+0200    INFO    zapio/writer.go:146     go: downloading github.com/josharian/intern v1.0.0
2023-10-23T12:19:34.757+0200    INFO    zapio/writer.go:146     go: downloading github.com/checkpoint-restore/go-criu/v5 v5.3.0
2023-10-23T12:19:34.772+0200    INFO    zapio/writer.go:146     go: downloading github.com/containerd/console v1.0.3
2023-10-23T12:19:34.795+0200    INFO    zapio/writer.go:146     go: downloading github.com/mrunalp/fileutils v0.5.0
2023-10-23T12:19:34.808+0200    INFO    zapio/writer.go:146     go: downloading github.com/opencontainers/selinux v1.10.0
2023-10-23T12:19:34.825+0200    INFO    zapio/writer.go:146     go: downloading github.com/vishvananda/netlink v1.1.1-0.20201029203352-d40f9887b852
2023-10-23T12:19:34.865+0200    INFO    zapio/writer.go:146     go: downloading golang.org/x/mod v0.13.0
2023-10-23T12:19:34.927+0200    INFO    zapio/writer.go:146     go: downloading github.com/minio/asm2plan9s v0.0.0-20200509001527-cdd76441f9d8
2023-10-23T12:19:34.927+0200    INFO    zapio/writer.go:146     go: downloading github.com/minio/c2goasm v0.0.0-20190812172519-36a3d3bbc4f3
2023-10-23T12:19:34.927+0200    INFO    zapio/writer.go:146     go: downloading github.com/klauspost/asmfmt v1.3.2
2023-10-23T12:19:34.941+0200    INFO    zapio/writer.go:146     go: downloading github.com/cncf/udpa/go v0.0.0-20220112060539-c52dc94e7fbe
2023-10-23T12:19:34.951+0200    INFO    zapio/writer.go:146     go: downloading github.com/syndtr/gocapability v0.0.0-20200815063812-42c35b437635
2023-10-23T12:19:34.952+0200    INFO    zapio/writer.go:146     go: downloading github.com/seccomp/libseccomp-golang v0.9.2-0.20220502022130-f33da4d89646
2023-10-23T12:19:34.967+0200    INFO    zapio/writer.go:146     go: downloading github.com/cilium/ebpf v0.7.0
2023-10-23T12:19:35.098+0200    INFO    zapio/writer.go:146     go: downloading github.com/vishvananda/netns v0.0.0-20200728191858-db3c7e526aae
2023-10-23T12:19:36.165+0200    INFO    zapio/writer.go:146     go: downloading github.com/Azure/azure-sdk-for-go/sdk/internal v1.3.0
2023-10-23T12:19:36.166+0200    INFO    zapio/writer.go:146     go: downloading github.com/AzureAD/microsoft-authentication-library-for-go v1.0.0
2023-10-23T12:19:36.166+0200    INFO    zapio/writer.go:146     go: downloading github.com/Azure/go-autorest/autorest/validation v0.3.1
2023-10-23T12:19:39.191+0200    INFO    zapio/writer.go:146     go: downloading github.com/gocql/gocql v1.3.2
2023-10-23T12:19:39.236+0200    INFO    zapio/writer.go:146     go: downloading github.com/DataDog/opentelemetry-mapping-go/pkg/internal/sketchtest v0.8.0
2023-10-23T12:19:39.243+0200    INFO    zapio/writer.go:146     go: downloading github.com/go-kit/kit v0.12.0
2023-10-23T12:19:39.244+0200    INFO    zapio/writer.go:146     go: downloading github.com/tj/assert v0.0.3
2023-10-23T12:19:39.248+0200    INFO    zapio/writer.go:146     go: downloading github.com/Netflix/go-env v0.0.0-20220526054621-78278af1949d
2023-10-23T12:19:39.249+0200    INFO    zapio/writer.go:146     go: downloading github.com/alibabacloud-go/darabonba-openapi/v2 v2.0.4
2023-10-23T12:19:39.250+0200    INFO    zapio/writer.go:146     go: downloading github.com/alibabacloud-go/sts-20150401/v2 v2.0.1
2023-10-23T12:19:39.251+0200    INFO    zapio/writer.go:146     go: downloading github.com/cloudflare/golz4 v0.0.0-20150217214814-ef862a3cdc58
2023-10-23T12:19:39.278+0200    INFO    zapio/writer.go:146     go: downloading github.com/frankban/quicktest v1.14.4
2023-10-23T12:19:39.278+0200    INFO    zapio/writer.go:146     go: downloading github.com/influxdata/line-protocol-corpus v0.0.0-20210922080147-aa28ccfb8937
2023-10-23T12:19:39.279+0200    INFO    zapio/writer.go:146     go: downloading github.com/kr/pretty v0.3.1
2023-10-23T12:19:39.288+0200    INFO    zapio/writer.go:146     go: downloading github.com/go-errors/errors v1.4.2
2023-10-23T12:19:39.289+0200    INFO    zapio/writer.go:146     go: downloading github.com/pingcap/errors v0.11.4
2023-10-23T12:19:39.292+0200    INFO    zapio/writer.go:146     go: downloading gopkg.in/check.v1 v1.0.0-20201130134442-10cb98267c6c
2023-10-23T12:19:39.309+0200    INFO    zapio/writer.go:146     go: downloading github.com/Masterminds/semver/v3 v3.1.1
2023-10-23T12:19:39.310+0200    INFO    zapio/writer.go:146     go: downloading github.com/onsi/ginkgo/v2 v2.11.0
2023-10-23T12:19:39.311+0200    INFO    zapio/writer.go:146     go: downloading github.com/onsi/gomega v1.27.10
2023-10-23T12:19:39.323+0200    INFO    zapio/writer.go:146     go: downloading github.com/alecthomas/assert/v2 v2.3.0
2023-10-23T12:19:39.325+0200    INFO    zapio/writer.go:146     go: downloading github.com/alecthomas/repr v0.2.0
2023-10-23T12:19:39.361+0200    INFO    zapio/writer.go:146     go: downloading github.com/jmespath/go-jmespath/internal/testify v1.5.1
2023-10-23T12:19:39.362+0200    INFO    zapio/writer.go:146     go: downloading github.com/alibabacloud-go/endpoint-util v1.1.0
2023-10-23T12:19:39.362+0200    INFO    zapio/writer.go:146     go: downloading github.com/alibabacloud-go/openapi-util v0.1.0
2023-10-23T12:19:39.362+0200    INFO    zapio/writer.go:146     go: downloading github.com/GoogleCloudPlatform/opentelemetry-operations-go/internal/cloudmock v0.44.0
2023-10-23T12:19:39.363+0200    INFO    zapio/writer.go:146     go: downloading github.com/alibabacloud-go/tea-utils/v2 v2.0.1
2023-10-23T12:19:39.364+0200    INFO    zapio/writer.go:146     go: downloading github.com/alibabacloud-go/tea v1.1.19
2023-10-23T12:19:39.364+0200    INFO    zapio/writer.go:146     go: downloading github.com/alibabacloud-go/alibabacloud-gateway-spi v0.0.4
2023-10-23T12:19:39.365+0200    INFO    zapio/writer.go:146     go: downloading github.com/alibabacloud-go/tea-xml v1.1.2
2023-10-23T12:19:39.366+0200    INFO    zapio/writer.go:146     go: downloading github.com/aliyun/credentials-go v1.1.2
2023-10-23T12:19:39.367+0200    INFO    zapio/writer.go:146     go: downloading github.com/joho/godotenv v1.3.0
2023-10-23T12:19:39.372+0200    INFO    zapio/writer.go:146     go: downloading github.com/jcmturner/goidentity/v6 v6.0.1
2023-10-23T12:19:39.379+0200    INFO    zapio/writer.go:146     go: downloading github.com/leesper/go_rng v0.0.0-20190531154944-a612b043e353
2023-10-23T12:19:39.398+0200    INFO    zapio/writer.go:146     go: downloading gotest.tools/v3 v3.2.0
2023-10-23T12:19:39.399+0200    INFO    zapio/writer.go:146     go: downloading github.com/alibabacloud-go/tea-utils v1.3.1
2023-10-23T12:19:39.402+0200    INFO    zapio/writer.go:146     go: downloading github.com/kr/text v0.2.0
2023-10-23T12:19:39.402+0200    INFO    zapio/writer.go:146     go: downloading github.com/rogpeppe/go-internal v1.11.0
2023-10-23T12:19:39.405+0200    INFO    zapio/writer.go:146     go: downloading github.com/jackc/pgmock v0.0.0-20210724152146-4ad1a8207f65
2023-10-23T12:19:39.406+0200    INFO    zapio/writer.go:146     go: downloading github.com/cockroachdb/apd v1.1.0
2023-10-23T12:19:39.408+0200    INFO    zapio/writer.go:146     go: downloading github.com/go-logr/zapr v1.2.4
2023-10-23T12:19:39.412+0200    INFO    zapio/writer.go:146     go: downloading github.com/hashicorp/consul/sdk v0.14.1
2023-10-23T12:19:39.413+0200    INFO    zapio/writer.go:146     go: downloading github.com/dnaeon/go-vcr v1.2.0
2023-10-23T12:19:39.428+0200    INFO    zapio/writer.go:146     go: downloading github.com/go-test/deep v1.1.0
2023-10-23T12:19:39.431+0200    INFO    zapio/writer.go:146     go: downloading github.com/hexops/gotextdiff v1.0.3
2023-10-23T12:19:39.437+0200    INFO    zapio/writer.go:146     go: downloading github.com/containerd/typeurl v1.0.2
2023-10-23T12:19:39.441+0200    INFO    zapio/writer.go:146     go: downloading github.com/Azure/go-autorest/autorest/mocks v0.4.2
2023-10-23T12:19:39.442+0200    INFO    zapio/writer.go:146     go: downloading github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/storage/armstorage v1.2.0
2023-10-23T12:19:39.458+0200    INFO    zapio/writer.go:146     go: downloading github.com/Azure/go-autorest/autorest/azure/auth v0.4.2
2023-10-23T12:19:39.460+0200    INFO    zapio/writer.go:146     go: downloading github.com/golang/glog v1.1.0
2023-10-23T12:19:39.461+0200    INFO    zapio/writer.go:146     go: downloading github.com/HdrHistogram/hdrhistogram-go v1.1.2
2023-10-23T12:19:39.461+0200    INFO    zapio/writer.go:146     go: downloading github.com/dimfeld/httptreemux v5.0.1+incompatible
2023-10-23T12:19:39.465+0200    INFO    zapio/writer.go:146     go: downloading github.com/shoenig/test v0.6.6
2023-10-23T12:19:39.467+0200    INFO    zapio/writer.go:146     go: downloading github.com/clbanning/mxj/v2 v2.5.5
2023-10-23T12:19:39.480+0200    INFO    zapio/writer.go:146     go: downloading github.com/tjfoc/gmsm v1.3.2
2023-10-23T12:19:39.487+0200    INFO    zapio/writer.go:146     go: downloading github.com/alibabacloud-go/debug v0.0.0-20190504072949-9472017b5c68
2023-10-23T12:19:39.492+0200    INFO    zapio/writer.go:146     go: downloading github.com/jarcoal/httpmock v1.3.0
2023-10-23T12:19:39.492+0200    INFO    zapio/writer.go:146     go: downloading github.com/maxatome/go-testdeep v1.12.0
2023-10-23T12:19:39.506+0200    INFO    zapio/writer.go:146     go: downloading github.com/grpc-ecosystem/go-grpc-middleware v1.4.0
2023-10-23T12:19:39.506+0200    INFO    zapio/writer.go:146     go: downloading github.com/bitly/go-hostpool v0.0.0-20171023180738-a3a6125de932
2023-10-23T12:19:39.508+0200    INFO    zapio/writer.go:146     go: downloading github.com/bmizerany/assert v0.0.0-20160611221934-b7ed37b82869
2023-10-23T12:19:39.517+0200    INFO    zapio/writer.go:146     go: downloading github.com/hashicorp/go-msgpack v0.5.5
2023-10-23T12:19:39.517+0200    INFO    zapio/writer.go:146     go: downloading github.com/hashicorp/memberlist v0.5.0
2023-10-23T12:19:39.519+0200    INFO    zapio/writer.go:146     go: downloading github.com/dgryski/go-farm v0.0.0-20200201041132-a6ae2369ad13
2023-10-23T12:19:39.521+0200    INFO    zapio/writer.go:146     go: downloading github.com/montanaflynn/stats v0.7.0
2023-10-23T12:19:39.531+0200    INFO    zapio/writer.go:146     go: downloading github.com/oklog/ulid v1.3.1
2023-10-23T12:19:39.535+0200    INFO    zapio/writer.go:146     go: downloading github.com/mitchellh/go-testing-interface v1.14.1
2023-10-23T12:19:39.538+0200    INFO    zapio/writer.go:146     go: downloading github.com/dimchansky/utfbom v1.1.0
2023-10-23T12:19:39.538+0200    INFO    zapio/writer.go:146     go: downloading github.com/Azure/go-autorest/autorest/azure/cli v0.3.1
2023-10-23T12:19:39.538+0200    INFO    zapio/writer.go:146     go: downloading github.com/pascaldekloe/goe v0.1.0
2023-10-23T12:19:39.559+0200    INFO    zapio/writer.go:146     go: downloading gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7
2023-10-23T12:19:39.561+0200    INFO    zapio/writer.go:146     go: downloading github.com/gorilla/handlers v1.5.1
2023-10-23T12:19:39.562+0200    INFO    zapio/writer.go:146     go: downloading github.com/zeebo/assert v1.3.0
2023-10-23T12:19:39.570+0200    INFO    zapio/writer.go:146     go: downloading github.com/VividCortex/gohistogram v1.0.0
2023-10-23T12:19:39.574+0200    INFO    zapio/writer.go:146     go: downloading github.com/sean-/seed v0.0.0-20170313163322-e2103e2c3529
2023-10-23T12:19:39.574+0200    INFO    zapio/writer.go:146     go: downloading github.com/hashicorp/go-sockaddr v1.0.2
2023-10-23T12:19:39.574+0200    INFO    zapio/writer.go:146     go: downloading github.com/google/btree v1.0.1
2023-10-23T12:19:39.606+0200    INFO    zapio/writer.go:146     go: downloading github.com/google/pprof v0.0.0-20230705174524-200ffdc848b8
2023-10-23T12:19:39.606+0200    INFO    zapio/writer.go:146     go: downloading github.com/go-task/slim-sprig v0.0.0-20230315185526-52ccab3ef572
2023-10-23T12:19:39.703+0200    INFO    zapio/writer.go:146     go: downloading github.com/clbanning/mxj v1.8.4
2023-10-23T12:19:40.141+0200    INFO    builder/main.go:128     Getting go modules
2023-10-23T12:19:40.141+0200    INFO    builder/main.go:25      Running go subcommand.  {"arguments": ["mod", "download"]}
2023-10-23T12:20:00.706+0200    INFO    builder/main.go:93      Compiling
2023-10-23T12:20:00.706+0200    INFO    builder/main.go:25      Running go subcommand.  {"arguments": ["build", "-trimpath", "-o", "otelcol-contrib", "-ldflags=-s -w"]}
2023-10-23T12:20:16.854+0200    INFO    builder/main.go:112     Compiled        {"binary": "./_build/otelcol-contrib"}
```

</details>

<details>
<summary>contrib v0.87.0 manifest, re-run (all modules cached)</summary>

```
 ❯ ./builder --config manifest.yaml 
2023-10-23T12:21:52.093+0200    INFO    internal/command.go:117 OpenTelemetry Collector Builder {"version": "dev", "date": "unknown"}
2023-10-23T12:21:52.099+0200    INFO    internal/command.go:153 Using config file       {"path": "manifest.yaml"}
2023-10-23T12:21:52.100+0200    INFO    builder/config.go:106   Using go        {"go-executable": "/home/pablo.baeyens/.gvm/gos/go1.20/bin/go"}
2023-10-23T12:21:52.107+0200    INFO    builder/main.go:82      Sources created {"path": "./_build"}
2023-10-23T12:21:52.107+0200    INFO    builder/main.go:25      Running go subcommand.  {"arguments": ["mod", "tidy", "-compat=1.20"]}
2023-10-23T12:21:53.067+0200    INFO    builder/main.go:128     Getting go modules
2023-10-23T12:21:53.067+0200    INFO    builder/main.go:25      Running go subcommand.  {"arguments": ["mod", "download"]}
2023-10-23T12:21:53.372+0200    INFO    builder/main.go:93      Compiling
2023-10-23T12:21:53.372+0200    INFO    builder/main.go:25      Running go subcommand.  {"arguments": ["build", "-trimpath", "-o", "otelcol-contrib", "-ldflags=-s -w"]}
2023-10-23T12:21:55.574+0200    INFO    builder/main.go:112     Compiled        {"binary": "./_build/otelcol-contrib"}
```

</details>

Reply:
If configurable, my vote is on making this opt-out since I would expect for most users there would be some caching (either local or on CI). Does that make sense to you?

Reply:
Right, I guess my preference was on opt-in as it would continue working as it is today. In general, the information is not particularly interesting unless i'm troubleshooting an issue, in which case i would want to run the builder with --verbose

Reply:
Thanks @mx-psi!